feat(content): Tavish & Kormak - Human Mission (#10564)
endless-sky/endless-sky: data/human/human missions.txt

			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

mission "Tavish and Kormak 1"
	name "Tavish and Kormak"
	description "Take Tavish and his friend Kormak, who are about to make a monumental discovery on <origin>, to <destination> to pick up some mining tools. They're not going to pay."
	source "New India"
	destination "Clark"
	passengers 2
	to offer

> 		random < 30

Author: What do you think it should be?

> 		random < 30

Reviewer: I don't know. Maybe `random < 50`. Although, maybe what you have works too. I guess it depends on how easy you want it to be to get it.

> 		random < 30

Author: Maybe the third time they land on the planet?

> 		random < 30

Reviewer: Maybe... but that would essentially be a `random < 33`, and overcomplicate things.

	on offer
		conversation
			`You make your way through the makeshift spaceport, breathing in the thick humid air and feeling the relentless glare of New India's twin suns. The muddy streets are filled with a constant bustling stream of humans, animals, and vehicles fighting their way to either unload the raw materials of this world, or load up on the precious goods needed to tame it.`
			`	You notice a small uncrowded cafe and bakery slightly off the beaten path where you can get some respite from the crowds and the sun.`
			choice
				`	(Take a seat in the cafe.)`
					goto seat
				`	(Head back to your ship.)`

> 			`	You see no reason to stick around this rudimentary spaceport so you make your way back to your ship.`

Author: I've removed one of them. Two of them are exclusive of each other. So there's only twice you'd see them.

				defer

			label seat
			`	You take a seat near a window that does a surprisingly good job at muffling the noises beyond it. The cafe is quaint, nearly empty, and cool. Initially you're the only patron here, but after moment a pair of men enter.`
			choice
				`	(Check the menu.)`

> 				`	(Get up to leave.)`

Author: I agree with your first comment. But I don't like to remove the choice here. I think one think you're not seeing with that I am doing is I use choices to break up scenes, large text blocks and help create spacing in the story. 

The choice here shifts the scene from you being in the diner, to an enigmatic conversation with Tavish and Kormak. When I remove it.. it all bleeds together. Lessens the impact of the scene.

> 				`	(Get up to leave.)`

Reviewer: Hmm... maybe if a waiter handed you a menu and you had a choice to order something on it or not?

			`	"... and you, my good fellow, have the look of someone who is just about to become of one our friends." Tavish flashes you with a wide smile.`
			`	"You see, me and my fellow scholar are in desperate need of your eminent services. We are on the very cusp of a monumental discovery here in the untamed wilds of New India." Tavish leans in and gestures wildly with his hands. "The sort of discovery certain to rewrite the entire history of this sector! Why, science as we know it will never be the same."`
			`	"Never be the same," Kormak's gruff voice punctuates and regurgitates Tavish's statement.`
			`	"Yes! Yes! But first we need a capable captain such as yourself to ferry us over to Clark to pick up our equipment and bring us back here. A most trivial request for one of your talents, so trivial in fact you might even be inspired to waive your normal fee and do it pro bono; for science that is."`
			`	"For science." Kormak echoes blandly.`
			choice
				`	"I'll do it, but not for free."`
				`	"Surely you have some money."`
				`	"Fine, I'll do it for free."`
					goto forscience
				`	"For science!"`

> 					goto forscience

Author: Good call. Added it.

			`	"We are broke." Kormak helpfully adds.`
			`	"It is true," Tavish sighs. "Our circumstances are most unfortunate; we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
			`	"We asked everyone." Kormak says.`
			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? We're sure our fortunes are about to change! So what do you say, my good fellow? Are you ready join us on our grand adventure and ferry us to the hardscrabble world of Clark?"`
			choice
				`	"My name is <first> <last>, not Mayflower. You have no idea who I am, do you?"`
				`	"I am not Mayflower. My name is Captain <last>."`
				`	"You have mistaken me for someone else. I am Captain <last>, not Mayflower."`

> 				`	"Uh, yes?"`

Author: That is an option, but in the next line. I feel like before you'd tell them no, you'd first tell you're not who they think you are.

> 				`	"Uh, yes?"`

Reviewer: I guess that works.

					goto reluctantlyforscience
				`	"For science!"`
					goto reluctantlyforscience
				`	"No, I only work for paying customers."`
				`	"No."`

			`	Tavish's bright eyes turn wet, and he drops his gaze to the table before you. Slowly, as if he's in actual pain, he places his bowler hat back atop his head. He then stands up, promptly followed by the lumbering Kormak.`
			`	"Very well. You're not the first captain to disdain science and cast us out into the streets, but I had truly hoped you'd be the first to see reason. I bid you good day."`
			`	As Tavish turns to leave, Kormak waivers for a just a moment. "Not like mirror," he finally says, and then follows Tavish out.`
				decline
				
			label reluctantlyforscience
			`	Tavish breaks into a wide grin and mock punches the air just above his head. "Huzzah, my good chap! Yes, that is the pioneering spirit!"`
			`	He abruptly turns to Kormak and they share a few congratulatory handshakes.`
			`	Once the handshaking concludes Tavish carefully rises from his seat, "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice

> 				`	"The <ship>."`

Author: Sometimes a choice is a nice way to break up dialogue.

> 				`	"The <ship>."`

Reviewer: It is your choice, of course, but personally I find that this kind of single-choice dynamic makes the game feel more on-rails. When the game's speaking for you, it kind of gets hidden in the background and glossed over, whereas when you're given a choice that has no impact on the story it's more obvious that the game is choosing what you say and do.

			`	"We are broke." Kormak helpfully adds.`
			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
			`	"We asked everyone." Kormak says.`
			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible and we shall commence with our grand adventure to the hardscabble world of Clark!"`
			choice
				`	"Yes! To Clark! Although, my name is not Mayflower, it's <last>.`
				`	"Fine, but I better not regret this. Also, my name is <last>, not Mayflower."`

> 				`	"Sure, whatever, but where did you get Mayflower from? I'm Captain <last>."`

Author: There is an option if your name is Mayflower. Line 7513, I think it skips the above options.

> 				`	"Sure, whatever, but where did you get Mayflower from? I'm Captain <last>."`

Reviewer: Oops, yes, you're right.

			`	"Oh, how mortifing! I..." Tavish's voice drops off as he appears to be, finally, at a loss for words."`
			`	"Looked in wrong manifest." Kormak seems to be suffering none of Tavish's tongue-tied embarrassment.`
			`	"Er, yes, about that.." there's a pause before Tavish, his face red, finally says, "We may have been a tad bit desperate, and have been asking every captain who lands here, regardless of reputation." `
			`	"We asked everyone." Kormak repeats his comment from earlier.`
			`	"I must have gotten the flight manifest confused with yours and a different ship. I am truly apologetic for misleading you. To be frank we have no idea who you are, Captain <last>."`
			`	"None." Kormak adds.`
			`	Tavish holds up a finger. "But, we can already tell you are a chap of quality, and the possesor of a keen curious mind. The sort of fellow we can put our absolute faith in. In fact, I see a lot of ourselves in you, truly we are kindred spirits."`
			`	"Like looking in mirror." Kormak says.`
			`	Tavish suddenly stands. "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice

> 				`	"The <ship>."`

Author: As above, I prefer to leave this in.

			`	Todd looks down at Tavish's hands, his face full of conflicting emotions before at last a determined resolve washes over him and he nods firmly, "Of course, Dean-Praetor. I am sure the executives at SpaceFork HQ would not want me to hinder your important work."`
			`	With that he promptly turns and leads the three of you past a long line of storage containers. "Here it is."`
			`	He taps the keypad unlocking the door, awkwardly salutes Tavish, and then marches away with a fresh air of confidence.`
			`	Tavish and Kormak enter the container.`
			choice
				`	(Follow them in.)`
			`	"By Skoll's Fang!" Kormak rumbles.`
			`	There are torn suitcases, clothes and various personal objects strewn throughout the container. The place has been ransacked and the equipment appears to be missing.`
			`	Tavish rubs his nose and paces back and forth.`
			`	"Stole it." Kormak says dryly.`
			`	"Yes, yes, I know. I suppose we should have accompanied him. Don't say it!" Tavish says with irritation.`

> 			`	"Told you so." Kormak says defiantly.`

Reviewer: ```suggestion
			`	"Told you so," Kormak says defiantly.`
```

			`	The area around you is momentarily frozen, shocked by Kormak's quick and easy violence. "Tell us what we need. And this man will be fine. Call for help, or refuse us, and this will get messy for everyone."`
			`	The guard is so red in the face it looks like he's about to pass out. "O-okay. Just don't hurt anyone, please!" The official moves to her terminal, strikes her keyboard for a few moments and then looks back up. "It s-says he's been impounded multiple times by for selling contrband from Deadman's Cove. Th-that's it." `
			`	Kormak lets the now unconscious guard's body slump to the floor. "Thank you, ser." Kormak tips his hat to her and turns to Tavish.`
			`	Tavish sighs, "Splendid work, though I do think we should run now."`
			`	He turns to you. "We'll meet you at your ship."`
			`	The two of them then sprint off before the authorities have time to respond.`
			choice

> 				`	(Head back to your ship.)`

Author: It feels odd to me to force the player into action like that. I can think of many games I've played where there is no choice, only one path to take, and the game makes me take that path. So, even though I have no agency, I am still in taking the action.

> 				`	(Head back to your ship.)`

Reviewer 2: Perhaps you could add a "Wander around the spaceport for a while" option? Still leads to the same dialog eventually, but tells a little less forced?
~~definitely not saying this because I like your writing~~

			choice
				`	"Fine, but first you need to answer some question."`
					goto questions
				`	"Nope. I don't know what this is, but I am out."`

			label quit
			`	Tavish frowns and sighs, but says nothing. He looks at Kormak and then begins to wander away.`
			`	Kormak glares at you and then says, "Could have had fun."`
			`	The large man turns and follows Tavish back into Clark's wild spaceport.`
				decline
			
			label questions
			`	"Of course." Tavish nods emphatically, "We assuredly are an open book."`
			`	Kormak delivers a dissenting grunt.`
			`	"Uh, mostly open." Tavish quickly amends.`

			label questions2
			choice
				`	"Just who exactly are you?"`
					goto who1
					to display
						not "who1"
				`	"What sort of science are you doing on New India?"`
					goto india1
					to display
						not "india1"
				`	"What is this equipment you need so badly?"`
					goto equipment1
					to display
						not "equipment1"
				`	"I have no more questions. But from now on I'm in charge. Now, let's go to Deadman's Cove."`
					goto end

			label who1
			action

> 				set "who1"

Author: I based them off names for conditions Bene used in a previous mission we worked together on. Since I cancel them, I don't think their names matter?

			`	"Just as I said earlier, we're Tavish and Kormak." Tavish starts.`
			`	"Kormak and Tavish." Kormak corrects him.`
			`	"And," Tavish says ignoring Kormak, "we are doctors, researchers, and most importantly, explorers!"`
			`	Tavish pauses and levels a look at Kormak, "At least, I think we're both doctors."`

> 			`	"Very small school in Deep. Likely not heard of." Kormak says nonchalantly.`

Author: He's from the Deep. Not a planet called Deep.

> 			`	"Very small school in Deep. Likely not heard of." Kormak says nonchalantly.`

Reviewer: After reading more and seeing Kormak's speech patterns, "in Deep" works (although because there is a planet called Deep, it's a little confusing). I'd still use the comma, though:
```suggestion
			`	"Very small school in Deep. Likely not heard of," Kormak says nonchalantly.`
```

			choice
				`	"You're not the 'Dean-Praetor' of the Oxford-Harvard Science and Research Institute, though, are you?"`
				`	"I am pretty sure you lied to that SpaceFork employee about being a 'Dean-Praetor'."`

> 				`	"Do Dean-Praetor's of Oxford-Harvard normally go around having their colleagues mug security guards?"`

Author: I think it works. Perceived choice isn't a bad thing. A choice, even when there isn't one, also forces the player out of passivity when engaging.

			`	"Monsters. Elves. Dragons. Squirrels." Kormak says.`
			`	"Although we are not entirely indepedant, oh no. We are proud members of, and modestly funded by, the Society of Gentlemanly Exemplary Exploration, which was founded by the esteemed Dr. Henry Forsyth who cured the Rigellian Plague. SOGEE for short."`
			`	"Soggy." Kormak pronouces.`
			`	Tavish makes a pained expression as he says, "It is a most unfortunate pronunication for our esteemed society, but yes, SOGEE is what most call it. Hopefully this has satisfactorily answered your question?"`
				goto questions2
			
			label india1
			action
				set "india1"
			`	Tavish clears his throat, and then says quietly, "New India is a very recently colonized world. It is filled with life, and the remnants of life that have yet to be discovered."`
			`	He leans in and says even more quietly, "On a recent expedition Kormak and I discovered what we believe to be remains of a crashed ship. A xeno-ship."`
			`	"Aliens." Kormak adds helpfully.`
			`	Tavish straightens up, "That is what we're after, and why we need your help! So you see you really are on the cutting edge of discovery with us. A fellow adventurer and explorer!"`

> 			`	"Soggy." Kormak affirms.`

Author: it's meant to be there.

				goto questions2
			
			label equipment1
			action
				set "equipment1"
			`	"The deep jungles of New India are filled with aggressive tree-analogs." Tavish explains.`

> 			`	"Monsters." Kormak says with bared teeth.`

Reviewer: ```suggestion
			`	"Monsters," Kormak says with bared teeth.`
```

			`	"Unlike our more civilized trees on Earth, these ones are significantly more mobile. They will steal and smother anything left out, including pack animals and people."`
			`	Kormak removes his cap and says solemnly, "Gerald."`
			`	Tavish nods somberly, "Indeed, poor Gerald. If only his legs were as quick as his mind."`

> 			`	"More cardio." Kormak agrees.`

Reviewer: ```suggestion
			`	"More cardio," Kormak agrees.`
```

			`	"We had a special mechanical ring made to surround our campsite in ignited heat and prevent the tree-analog's tendrils from entering or interfering with our work."`

> 			`	"Firewall." Kormak adds.`

Reviewer: ```suggestion
			`	"Firewall," Kormak adds.`
```

			`	"There was also a drill and sifter specially designed and programmed by Dr. Gerald to help uncover the vessel without damaging it," Tavish continues, "and, lastly, an underground sonar headset that would allow us to see its buried structure and avoid injuring it during our excavation."`
			`	Tavish shrugs. "There were other things like tents, and picks, and shovels and what not. But all of that we can get on world. The ring, drill, and headset are all irreplaceable and absolutely necessary for this dig."`

> 			`	"Irreplaceable." Kormak confirms.`

Reviewer: ```suggestion
			`	"Irreplaceable," Kormak confirms.`
```

				goto questions2

			label end
			action
				clear "who1"
				clear "india1"

> 				clear "equipment1"

Reviewer: Ah, I see how this works. These condition names might be okay, then, but I'd still make them more descriptive if possible.

Author: But why?

	destination "Hephaestus"
	passengers 2
	to offer
		has "Tavish and Kormak 2: done"
	on offer
		conversation
			`Hospitality is in short supply as most of the pirates and free traders you pass either ignore you entirely or give you menacing glares. The spaceport bar on the archipelago's main island functions as a sort of catchall for services you'd expect at a larger and better organized spaceport. There's various fixers, vendors, mercenaries, and opportunists plying their trades in corners, booths and tables.`
			`	Tavish makes a beeline for one of the bartenders, the closest thing to any sort of identifiable administrator, and waves him down.`
			`	"What'll it be, your lordship?" the bartender asks mockingly after seeing Tavish's overly formal getup.`
			`	"I'll take a sidecar, my good man, and my large companion will have a Scottish Ale, assuming you have any." Tavish says brightly.`
			`	The bartender pours two glasses of rum and pushes them to Tavish. "My apologies, me lord," he gives a ridiculous bow, "but we seem to be out of your favored swill."`
			`	The bartender flashes Tavish a yellow grin that's missing a few key teeth. "Anything else I can getchya?"`

> 			`	"Information." Kormak grumbles, stepping closer to the bartender, who despite Kormak's size does not flinch.`

Reviewer: ```suggestion
			`	"Information," Kormak grumbles, stepping closer to the bartender, who despite Kormak's size does not flinch.`
```

			`	Instead the bartender's smile only widens to give everyone a better look at his bloody gums, "Now I's might just know a thing or two. But it'll cost you a sight more than a couple glasses of rum."`
			`	Tavish holds up his glass and looks in it, as if deciding if he's willing to drink it or not. "We need to find a ship that frequents this port of call."`
			`	The bartender taps his overgrown moldy nails on the bar, "Ah yes, me lord. I can probably do that. For a mere 50,000 credits, that is."`
			`	Tavish flinches at the number and then turns to look at you, "Your call."`
			label options
			choice
				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
					goto tav1
				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
					goto tav2
					to display

> 						has "kormak1"

Reviewer: Why is this repeated?

Author: I am not sure I understand the question.

Reviewer: You have the same choice in here twice, and so they will both display if you have the condition `kormak1`

			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 30,000 credit buy in, plus the information on our first unwitting mark, and you're all set to be rich."`
			`	"I'm in!" The bartender can't move fast enough, and immediately begins transferring the funds. "Who are you looking for?"`
			choice
				`	"We're looking for Hans Fuller of the Broken Record."`
			action
				payment 30000
			`	"Ha, that poor sod," the bartender can't help but chuckle to himself. "Last I was hearing, he was snatched up by the Syndies. By now he's probably on his way back to his hole on Buccaneer Bay. I'd look for 'em around the Persian system."`
			`	Tavish thanks the man and promises him he'll be seeing his first returns in the near future, and with that the three of you head out of the bar.`

			label story
			choice
				`	"Tavish, what's your story?"`
				`	"Tavish that was amazing! Where did you learn to do that?"`

> 				`	"Tell me the truth, Tavish. Are you conning me too?"`

Reviewer: I'd give a choice to skip this, and cut out at least one of the options here.

Author: Skip your ability to ask him what his story is after he talks a dude into paying you for the information you need?

Reviewer: Yes. This is great for character development, but at the end of the day not every player cares about player development or is playing a captain who would ask nosy questions about another character.

Author: Done.

			`	He shakes his head sadly, "In order to survive in that environment I had to become very good at working with those around me. Make them believe we were on the same side, believed the same things, had the very same lofty goals. I got quite good at it."`
			`	His voice becomes somber, "Perhaps too good. I started to forget where the genunine Tavish began, and the counterfeit Tavish ended. So, I stopped hiding who I was and made my interests and studies known to all!"`
			`	A wry grin spreads across his face, "And was promptly thrown out! So, now I am here with you and Kormak following my passion, on the forefront of discovery and adventure! And I am not too disconcerted that my old talents have some use."`
			`	After you arrive back at your ship, Tavish says, "Now we shall see a man in Hephaestus about some misplaced equipment," and hops aboard.`
				accept
		
			label kormak1
				action "kormak1"
			`	Kormak cracks a fist and steps right up to the bar. With deft movement Kormak has the barkeeper's neck in his veiny hands.`

> 			`	"Information." Kormak says.`

Reviewer: ```suggestion
			`	"Information," Kormak says.`
```

			`	With him gone the fighting dies down. Eventually the bartender gathers himself up well enough to look you in the eye and say, "Now it's 100,000 credits for that information."`
				goto options
	
			label tav2
			action
				clear "kormak1"
			`	Tavish steps around the broken bar bottles and stools and gives the bartender his winningest smile.`
			`	"My good man, you seem like a discerning fellow. Always looking to make a deal with an eye out for profit, am I right?"`
			`	The bartender looks piqued and slowly nods, "Aye."`
			`	"You're probably wondering what a pompous overdressed man like me is doing in your quaint out-of-the-way boutique. Well, I was looking for just the sort of enterprising soul who knows a thing or two about making a shilling."`
			`	Tavish proceeds to describe a long and complex scam utilizing unwitting pirates, corrupt Syndicate trade officials, and fabulously wealthy marks from the paradise world. As he talks the barkeep leans further and further in, his eyes widening beneath a growing devious grin."`

> 			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 25,000 credit buy in, plus the information on our first unwitting mark, and you're all set to be rich."`

Reviewer: ```suggestion
			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 25,000 credit buy-in, plus the information on our first unwitting mark, and you're all set to be rich."`
```

> 			`	"The bar tender rubs his still red neck. "Way I see it, lord, is that you still owe me for all this mess. How about this. My buy in is free, and I'll tell you where you can find your man?"`

Reviewer: ```suggestion
			`	"The bartender rubs his still red neck. "Way I see it, lord, is that you still owe me for all this mess. How about this. My buy-in is free, and I'll tell you where you can find your man?"`
```

> 			`	Tavish shrugs, "Like I said! A discerning fellow! We agree, my associate will now provide you with the details."`

Reviewer: ```suggestion
			`	Tavish shrugs, "Like I said! A discerning fellow! We agree; my associate will now provide you with the details."`
```

				accept

	npc board
		government "Bounty (Disguised)"
		system "Persian"
		personality entering staying target timid uninterested
		ship "Modified Argosy" "Broken Record"

		conversation
			`With the crew subdued you scour the Broken Record's cargo hold. Unfortunately, you find no trace of Tavish and Kormak's scientific equipment.`
			`	Kormak drags the bruised and restrained Hans Fuller before you and Tavish.`
			`	"Salutations, Hans," Tavish says cheerfully.`
			`	Hans grunts uncomfortably, "Professor."`
			`	"Ah, lovely, you remember me. Perhaps you could also recall where our equipment is?"`
			`	"Where." Kormak's normally impassive voice is tinged with anger.`
			`	"I don't know what you're talking about! I never stole nothing. Let me go! I have my rights!"`
			`	Kormak growls and Tavish looks to you.`
			choice
				`	(Have Tavish get the information out of him.)`
					goto tavish
				`	(Have Kormak get the information out of him.)`
					goto kormak
				`	(Offer him 25,000 credits for the information.)`
					goto warbucks

			label tavish
			`	Tavish steps forward and leans in until his face is only a few inches from Hans.`
			`	"Do you remember Gerald?" Tavish asks.`

> 			`	"The slow guy? Yeah." Hans says.`

Reviewer: ```suggestion
			`	"The slow guy? Yeah," Hans says.`
```

			`	"Do you remember what happened to him?"`
			`	Hans pauses, then, "The trees got him."`
			`	Tavish straightens up. "Exactly, and unless you start talking. We're going to take you back to New India and feed you to those very same trees."`

> 			`	"Good plan." Kormak says with a savage gleam in his eyes.`

Reviewer: ```suggestion
			`	"Good plan," Kormak says with a savage gleam in his eyes.`
```

			`	Tavish leads you down one narrow corridor filled with endless ships. You don't see any labels or signage that make sense, but somehow Tavish plows on with a quick step and unshakable confidence.`
			`	Eventually he leads you into a large administrative complex. After four flights of stairs, three different elevators, and one very small trolley the three of you arrive at a bland corporate office with the words, "Repository of Confiscated Goods."`
			`	There are no doors, and a single window that looks like it was made of reinforced glass. The tired looking agent on the other side of the glass hits a button and his voices buzzes over to you, "What can I help you with?"`
			choice
				`	"We're looking for stolen scientific research equipment that was recently confiscated aboard the Broken Record."`
			`	The man looks down at his monitor and begins typing. After a while he stops and looks back up at you.`
			`	"Ah yes, I found it. You'll be happy to know we did indeed confiscate some unusual equipment from the Broken Record." He looks at you expectantly.`
			choice
				`	"Great! Can it be returned to us?"`
				`	"Finally. We're here to collect it."`

> 				`	"Hand it over. It belongs to us."`

Author: I am sorry, I don't see the problem here.

> 				`	"Hand it over. It belongs to us."`

Reviewer: I'd suggest maybe having the number of choices vary, so it isn't always three, but is somewhere in the 2-4 range. Also, I stand by my earlier statement that you could have a slightly tweaked version of the next paragraph for when the third choice gets selected.

Basically, my argument is that when the three choices are very similar (which they aren't here, so this occurrence actually works to some extent), they may as well be the same choice, at which point the perceived choice is kind of minimized because all the choices are the same.

> 				`	"Hand it over. It belongs to us."`

Author: Done

			`	The man gives you a pained expression, "I am sorry, but it is no longer on Hephaestus."`
			`	He says nothing else and waits.`
			choice
				`	"Okay. Where is it?"`
				`	"Tell me where it is, right now."`
				`	"Speak up! Where's our stuff?!"`
			`	The agent's smile does not reach his eyes, "I am sorry, but that information is confidential."`
			choice
				`	"How are we supposed to reclaim our stolen goods?"`
				`	"What do we have to do to get that information?"`

> 				`	"Help me out here, what am I supposed to do?"`

Author: They player doesn't know that they don't do anything. Also, this is a conversation you're having - not them.

> 				`	"Help me out here, what am I supposed to do?"`

Reviewer: The problem is that I believe this will just display one line of text at the top of the screen, then the menu, and hide the whole conversation you're having unless you scroll up.

			`	The agent sighs and presses a button and a form slips out of a crease in the wall and onto the floor in front of you, "That's a C-79 access for information pertaining to confiscated goods form. Fill that out with that attached processing fee of 20,000 credits and we will be able to release that information to you."`
			`	You turn to look at Tavish and Kormak who both give you hopeless shrugs.`

			label options
			choice
				`	(Have Tavish attempt to get the information out of the agent.)`
					goto tav
					to display
						not "tav"
				`	(Have Kormak attempt to get the information out of the agent.)`
					goto kor
					to display
						not "kor"
				`	(Fill out the form and pay the 20,000 credit processing fee.)`

> 					goto payup

Author: Good idea, done!

			`	Tavish swallows, "This is most disconcerting."`
			`	Without looking at you or Kormak he heads toward the exit.`
			`	"Bad history." Kormak says, before following Tavish out.`
			`	It seems Chiron is your next destination.`
	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish were on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

	on complete
		conversation
			`Your ship comes to a landing in a crowded spaceport. Some sort of commencement ceremony is going on and the place is packed with off-world family and friends who have come to watch the graduation.`
			`	It's obvious Tavish is uncomfortable here, but he does his best not to show it. He informs you that you'll need to get to the commuter rail deeper in the New Port spaceport to get to Hullen-Freeze.`

mission "Tavish Interlude"
	name "A chat"
	invisible
	landing
	source "Hephaestus"
	destination "Chiron"
	to offer
		has "Tavish and Kormak 3: done"
	on enter

	# You will see this conversation after the first jump you make.
	on enter
		conversation
			`	Tavish approaches you on the bridge, "Captain <last>, I'd like a word."`
			choice

> 				`	"Sure, what's up?"`

Author: No, it's a mistake =(

				`	"Talk."`
					goto talk
				`	"Not now, Tavish, I am busy."`
			`	Tavish nods, "Of course. I understand, Captain."`
			`	He turns and leaves you to your work.`
				decline

			label talk
			`	Tavish removes his bowler cap and holds it in his hands, "If we're going to confront Dr. Hullen-Freeze then there are a few things you should be made aware of." `
			`	Tavish fidgets with his hat, "We are former... acquaintances, and we did not leave things on the best of terms. Whatever he says, whatever he may say about me, I just want to ensure you Captain that I have been forthright with you, at least where it counts."`
			`	He pauses then says, "I am a member of the Society of Gentlemanly Exemplary Exploration, and Kormak and I are on the hunt for Xeno-Cryptids. I have not tried to con you on that matter, and I do believe our work is vital to science and exploration."`
			label options
			choice
				`	"What exactly is your relationship with Hullen-Freeze?"`
					goto relation
					to display
						not "relation"
				`	"Tell me more about the Society of Gentlemanly Exemplary Exploration."`
					goto soggy
					to display
						not "soggy"
				`	"Don't worry. I trust you."`

> 				`	"There's something about you I don't quite trust. But we've come this far and I intend to see this through."`

Author: Done.

	name "Enter Ashoka"
	description "Transport Tavish and Kormak to the dark continent of Ashoka on <planet>."
	source "New India"
	destination "New India"
	passengers 2
	cargo "Scientific Research Equipment" 5
	to offer
		has "Tavish and Kormak 5: done"
	on offer
		conversation
			`You find Tavish and Kormak outside a corner bakery with multiple duffel bags and rucksacks worth of gear.`
			`	Kormak takes a bite of pastry slathered in chocolate syrup.`
			`	"Have you tried the bread here, Captain <last>? It's surprisingly good." Tavish says as he pushes an empty plate away from him.`
			`	Kormak tries to say something but his mouth is too stuffed to make any sense of it. He too stands.`
			`	"The site is on the far side of the planet on the south-eastern continent they call Ashoka. Except for us and a few areiel surveys it's almost completely unexplored." Tavish licks his fingers and then pulls on two rucksacks front, and back, and grabs two duffle bags.`
			`	"Shall we go?" Tavish asks.`
			`	"Go." Kormak says as he picks up double the rucksacks and duffle bags.`
			choice
				`	"Let's go."`
				`	"Adventure awaits!"`
				`	"For Science!"`
				`	"Whatever."`

> 				`	"For Gerald!"`

Author: Done.

			`	Unphased by the incessant wailing Kormak gleefully hacks away at vines, branches and small trees as he leads you, Tavish and the small caravan of mobile equipment deeper into the terrifyingly unnatural forest.`
			`	By the end of the trek everyone is breathing heavily, and Kormak is covered in the blue sap or blood that leaks from the severed limbs of the trees.`
			`	Finally, Tavish announces that you've made it. Although it is hard for you to tell where exactly you've made it to. One acre of jungle looks remarkably like the next. The only distinguishing feature here being a black overgrown rock jutting out between two trees.`
			`	Kormak begins clearing out the area while Tavish unloads the gear.`
			choice
				`	(Help Tavish unload the gear.)`
					goto helptav
				`	(Help Kormak clear the area.)`
					goto helpkor
				`	(Sit and watch them work.)`

			`	This isn't your expedition so you take a seat and relax while Kormak and Tavish prepare the campsite.`
			`	A few hours pass and the twin suns of New India begin to set on the horizion. Kormak has returned to from hacking, and Tavish has begun making a campfire.`
				goto next
			
			label helptav

> 			`	You help Tavish set up the ring of fire and the tents within it.`

Author: Okay, see if you find that more clear?

				goto next
			
			label helpkor
			`	You follow Kormak has he hacks and slashes at various alien flora and pick up oozing remains he leaves behind.`
			`	"You did good." Kormak says without turning to look at you.`
			`	"Was important thing. Standing by Tavish. Honorable." He continues as trees crack before him.`
			`	"Skoll-Wolf ate my clan's sun, and the lying Dream Anuabhair brought them to new world with strange stars. It was hard, but my clan is hardy. They thought they tame strange world. But then monsters come, and everything changed. Where I from trust is all that keeps a man alive."`
			`	Kormak suddenly stops what he is doing to turn and look at you, "We have trust. You, Tavish, me. Remember that. And all worlds have monsters."`
			`	The large man turns from you and finishes chopping down the last wailing tree.`
			`	With the area around the designated campground clear he faces you and gives you the smallest nod, "Trust."`
			`	Over to the side Tavish has also finished his work and is making a campfire.`

> 			`	"Come." Kormak says.`

Reviewer: ```suggestion
			`	"Come," Kormak says.`
```

			label next
			choice

> 				`	(Take a seat around the campfire.)`

Author: I disagree.

> 				`	(Take a seat around the campfire.)`

Author: I still believe this is too important to remove. I added a second option though.

			`	As the night takes hold strange alien noises can be heard coming from the jungle. Tavish continues to tend the fire as Kormak starts roasting rations to pass around.`
			`	Tavish takes a seat and a bite of the roasted food that Kormak has put into his hand.`

> 			`	"Good expedition." Kormak says beneath the emerging stars.`

Reviewer: ```suggestion
			`	"Good expedition," Kormak says beneath the emerging stars.`
```

			`	Tavish nods, "Certainly more than any of us bargained for, but tomorrow the real work begins and who knows what wonders we will discovered in that ancient crashed ship."`

> 			`	"Yes." Kormak says between bites, "Where I from songs would be sung of our deeds."`

Reviewer: ```suggestion
			`	"Yes," Kormak says between bites. "Where I from songs would be sung of our deeds."`
```

					goto notathing

			`	Kormak stops chewing and looks hard at you for a long moment before finally says, "It is forbidden."`
			`	"Ah, about that," Tavish jumps in, "the thing is the Deep is more complex than most people realize. It is a society with three distinct layers."`
			`	Tavish mimes peeling a fruit, "The top layer is the one everyone knows about. It's clean, fair, democratic, and technologically advanced. It's the Deep you hear about in holo-dramas, and see talked about on the news. Most of the Deep is a part of this layer.`
			`	"And then," Tavish continues, "there's the middle layer. It is smaller and made up of people who practice what they call the old ways. Those who practice stone-funerals, or who pay some respect to the scattered God-Stones. It's people who still see themselves as the descendants of those first brave pilgrims that colonized the Deep. Like Hroar's Haulers."`
			`	Tavish pinches his fingers together like he's holding his invisible fruit's core, "Finally, there's the lowest layer. Few are aware of it, and I think it may be an embarrassment to leaders of the Deep. It's made up of people like Kormak, or the Ice-Witches of Nifel." `

> 			`	"The Ui Skathi." Kormak says, "Granddaughters of Skathi."`

Reviewer: ```suggestion
			`	"The Ui Skathi," Kormak says. "Granddaughters of Skathi."`
```

> 			`	"Was that what they called themselves?" Tavish shrugs, "Kormak's always a little sensitive about this. Anyways, they don't really make sense. How did they get to the Deep? Where did they come from? There's two prevailing theories. First, that there was a lost generational fleet that landed in the Deep earlier than anyone realized, that something went wrong, and that they degenerated in primitives."`

Reviewer: ```suggestion
			`	"Was that what they called themselves?" Tavish shrugs. "Kormak's always a little sensitive about this. Anyways, they don't really make sense. How did they get to the Deep? Where did they come from? There's two prevailing theories. First, that there was a lost generational fleet that landed in the Deep earlier than anyone realized, that something went wrong, and that they degenerated in primitives."`
```

			`	Kormak frowns, "Lies.`

> 			`	"Or, well this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`

Reviewer: ```suggestion
			`	"Or, and this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`
```

> 			`	"Not elves," Kormak interrupts with a hint of annoyance,"the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`

Reviewer: ```suggestion
			`	"Not elves," Kormak interrupts with a hint of annoyance, "the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`
```

			`	Kormak sighs and turns back to you, "If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf. Know that is my home."`

> 			`	"Enough about past. Only present matters." He says and then goes back to eating.`

Reviewer: ```suggestion
			`	"Enough about past. Only present matters." He goes back to eating.`
```

			`	Tavish laughs, "Of course. Let us discourse about less weighty matters."`

			label notathing

> 			`	You talk a little of unimportant and unremarkable things beneath the glittering twilight stars in the midst of an untamed alien jungle.`

Reviewer: ```suggestion
			`	You talk a little of unimportant and unremarkable things beneath the glittering twilight stars in the midst of an untamed alien jungle...`
```

			`	And then you head to bed.`

> 			choice

Reviewer: I'm not sure whether I like this choice or not. You definitely need some way to divide between days, so I think it works.

				`	(Sleep.)`

			`	You are awoken by something warm and wet slowly sliding up your leg. Your tent has been perforated, and you can tell the blazing ring of fire is off.`
			choice
				`	(Kick whatever is on your leg.)`

			`	With a few stiff kicks you force the grasping appendage to lets go of your leg and slithers back out the tent hole. You follow after it to see the entire campground is writhing in tenticle like vines.`
			choice
				`	(Shout for Tavish to do something!)`
					goto tavshout
				`	(Shout for Kormak to do something!)`
					goto korshout
				`	(Shout for everyone to do something!)`
		
			label tavshout
			`	Tavish lunges out of his tent in a robe and screams, "The damned trees have returned!"`
			`	He briefly kicks ineffectively at them before finding one that is trying to steal his machete and wrestling it free.`

> 			`	"Kormak sometimes has a point," he grunts as he mashes back the various tendrils, "words don't work against monsters."`

Reviewer: ```suggestion
			`	"Kormak sometimes has a point," he grunts as he mashes back the various tendrils. "Words don't work against monsters."`
```

				goto everyone

			label korshout
			`	Kormak flies out of his tent completely naked. His rippling body outlined in dark blue tattoos. He rips and tears the vines with his bare hands and teeth. Eventually he finds his axe being dragged away and liberates it from the vine. Then he tears into the wild foilage with fearsome berserker rage. The trees scream in his bloody wake.`

> 			`	"I am Gluniairn mac Kormakr!" He shouts into the night, "I am death!"`

Reviewer: ```suggestion
			`	"I am Gluniairn mac Kormakr!" He shouts into the night. "I am death!"`
```

			label everyone

> 			`	Everyone is now up and attacking the vines. Kormak is begins laughing and starts singing in his bloodlust:`

Reviewer: ```suggestion
			`	Everyone is now up and attacking the vines. Kormak begins laughing and starts singing in his bloodlust:`
```

> 			`	"Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
> 			`	Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
> 			`	Break before my battle axe, Flee before my thunder.`
> 			`	Die beneath a stranger's sky, and feed my clan's hunger."`

Reviewer: ```suggestion
			`	"Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
			`	"Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
			`	"Break before my battle axe, Flee before my thunder.`
			`	"Die beneath a stranger's sky, and feed my clan's hunger."`
```

You could probably also just add line breaks and treat it like a poem (sort of like block quotes in an essay)

Author: Whatever makes it look best?

Reviewer: Block-quote style would be:
```suggestion
			``
			`	Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
			`	Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
			`	Break before my battle axe, Flee before my thunder.`
			`	Die beneath a stranger's sky, and feed my clan's hunger.`
			``
```

			`	After a while nothing is moves. It seems all the alien vines have retreated back behind the tree line. Belatedly you realize they weren't terribly formidable. The vines were slow, weak, and soft.`
			choice
				`	"That's it? How did those things manage to kill Gerald?"`
					goto defeat
				`	"Truly we've won a glorious victory today."`
					goto victory
				`	"I am going back to bed.`
					goto wakeup

			label defeat
			`	Kormak and Tavish look at each other and then finally Tavish says, "He really was quite slow."`

> 			`	Kormak nods, "Very slow."`

Reviewer: ```suggestion
			`	Kormak nods. "Very slow."`
```

> 			`	"And he may have been aggressively afraid." Tavish adds.`

Reviewer: ```suggestion
			`	"And he may have been aggressively afraid," Tavish adds.`
```

			`	"Rolled up into ball," Kormak confirms. "Very sad. I kill trees in his memory."`
			`	Tavish solemnly agrees and then tinkers with the ring of fire for a moment. "Ah, here's the problem," he says, and switches it back on.`
			`	"We'll be completely safe." He says and heads back into his tent.`
			choice
				`	(Go back to sleep.)`
					goto wakeup

			label victory
			`	Both Kormak and Tavish cheer.`
			`	"You handled yourself quite well, Captain <last>." Tavish says. "Much better than Gerald."`

> 			`	Kormal gives you a solemn nod, "Very slow. Just rolled into ball. Said 'take me', and gone."`

Reviewer: American English

```suggestion
			`	Kormal gives you a solemn nod, "Very slow. Just rolled into ball. Said 'take me,' and gone."`
```

			`	Tavish nods, "We'll dedicate the paper we write on this expedition to his memory. Such are the risks we explorers must face."`
			`	"Risks." Kormak agrees.`
			`	Tavish tinkers with the ring of fire for a moment. "Ah, here's the problem," he says, and switches it back on.`
			`	"We'll be completely safe." He heads back into his tent.`
			choice
				`	(Go back to sleep.)`
			
			label wakeup
			`	The morning comes without further incident. You eat breakfast, trade a few more harrowing tales about your adeventures in the night and prepare to leave.`
			`	Tavish approaches you before you make the trek back to your ship. "I have one final request before you go, Captain <last>."`
			`	He nods back to jutting black rock he believes is part of an ancient ship. "We're going to spend two months here on our dig. If you could come back in that time to give us one final ride, we'd appreciate it."`

> 			`	"Appreciate it." Kormak echoes.`

Reviewer: ```suggestion
			`	"Appreciate it," Kormak echoes.`
```

			`	Tavish looks a little apprehensive, "Without your aid our predicament will be most perilous. Between the wreckage and our equipment I hypothesize we will have at least 90 tons of cargo to transplant. However, we are still low on funds. It may prove challenging to persuade another ship to transport us pro bono via radio. So please do come back in two months."`
			choice
				`	"I'll come back. Don't worry."`
				`	"We'll see. No promises."`
				`	"I do what I want."`
			`	Tavish nods, "In that case we've quite a bit of work to do, Captain. Take care."`
			`	He starts working on prepping the dig site, while Kormak escorts you back to your ship.`

> 			`	Kormak takes your hand in his, "May stone guide you." he says, and then disappears back into the jungle.`

Reviewer: ```suggestion
			`	Kormak takes your hand in his. "May stone guide you," he says, and then disappears back into the jungle.`
```

event "killhullen"

mission "Tavish and Kormak 6"
	name "Enter Ashoka"

> 	description "Transport Tavish and Kormak to daek continent of Ashoka on <planet>."

Reviewer 2: Daek?

Author: dark ><

Reviewer 2: ```suggestion
	description "Transport Tavish and Kormak to the dark continent of Ashoka on <planet>."
```

			label end
			`	The shadows turn a darker shade of purple as you leave the graveyard of the forgotten dead for the familiar confines of your ship.`
				decline

mission "Tavish and Kormak 1"
	name "Tavish and Kormak"

> 	description "A monumental discovery on <origin> is about to be pulled off by Tavish and Kormak. Take them to <destination> to pick up some mining tools. They won't pay."

Reviewer 2: Sorry, the original was my suggested description.
I believe the "monumental" in the new version would make it slightly too long. Maybe "key" instead?

			`	Todd again nods, this time with some vigor.`
			`	Tavish then launches into a monologue about systems and rules and how important the human element is to correcting those systems. At one point Tavish pulls out an ID and proclaims he's the "Dean-Praetor of the Oxford-Harvard Science and Research Institute on Chiron."`
			`	Finally, Tavish finishes his speech and places his hands on Todd's and looks the man square in the eyes. "Todd, my dear man, be the human element in the machine, and take us to our storage container. If not for me, than for science."`
			`	Todd looks down at Tavish's hands, his face full of conflicting emotions before at last a determined resolve washes over him and he nods firmly, "Of course, Dean-Praetor. I am sure the executives at SpaceFork HQ would not want me to hinder your important work."`
			`	With that he promptly turns and leads the three of you past a long line of storage containers. "Here it is."`
			`	He taps the keypad unlocking the door, awkwardly salutes Tavish, and then marches away with a fresh air of confidence.`
			`	Tavish and Kormak enter the container.`
			choice
				`	(Follow them in.)`
			`	"By Skoll's Fang!" Kormak rumbles.`

> 			`	There are torn suitcases, clothes and various personal objects strewn throughout the container. The place has been ransacked and the equipment appears to be missing.`

Reviewer 3: ```suggestion
			`	There are torn suitcases, clothes and various personal objects strewn throughout the container, and there's no sign of the equipment.`
```
Bit matter-of-factly... more showing would be nice

			`	Kormak delivers a dissenting grunt.`
			`	"Uh, mostly open." Tavish quickly amends.`

			label questions2
			choice
				`	"Just who exactly are you?"`
					goto who1
					to display
						not "who1"
				`	"What sort of science are you doing on New India?"`
					goto india1
					to display
						not "india1"
				`	"What is this equipment you need so badly?"`
					goto equipment1
					to display
						not "equipment1"
				`	"I have no more questions. But from now on I'm in charge. Now, let's go to Deadman's Cove."`
					goto end

			label who1
			action
				set "who1"
			`	"Just as I said earlier, we're Tavish and Kormak," Tavish begins.`
			`	"Kormak and Tavish," Kormak corrects him.`
			`	"And," Tavish says ignoring Kormak, "we are doctors, researchers, and most importantly, explorers!"`
			`	Tavish pauses and levels a look at Kormak, "At least, I think we're both doctors."`
			`	"Very small school in Deep. Likely not heard of," Kormak says nonchalantly.`
			choice
				`	"You're not the 'Dean-Praetor' of the Oxford-Harvard Science and Research Institute, though, are you?"`
				`	"I am pretty sure you lied to that SpaceFork employee about being a 'Dean-Praetor'."`

> 				`	"Do Dean-Praetor's of Oxford-Harvard normally go around having their colleagues mug security guards?"`

Reviewer 3: ```suggestion
				`	"Do Dean-Praetors of Oxford-Harvard normally go around having their colleagues mug security guards?"`
```

> 			`	"No." Tavish admits, "I am not really the Dean-Praetor of Oxford-Harvard."`

Reviewer 3: ```suggestion
			`	"No," Tavish admits, "I am not really the Dean-Praetor of Oxford-Harvard."`
```

			`	"Irreplaceable," Kormak confirms.`
				goto questions2

			label end
			action
				clear "who1"
				clear "india1"
				clear "equipment1"
			`	Tavish brightens as he says, "I knew an erudite fellow such as yourself would see the value of our work. And for now on we will check with you before we do anything too untoward, right, Kormak?"`
			`	The red-haired giant nods, "Ask first."`
			`	"Grand," Tavish claps. "Now, let's see what we can discover at <planet>."`
				accept

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
	on complete
		conversation
			`The foam surf crashes against the island only a few short meters from where your ship sets down. The thundering waves make it hard to hear anything as you step outside. If you're going to have any hope of finding Hans Fuller and the stolen equipment your best bet will be the spaceport. Hopefully the sea will be a little quieter there.`

> mission "Tavish and Kormak 3"

Reviewer 3: It's half past midnight, I'll do more... sometime soon hopefully. Here's where I stopped.

			label end
			`	The shadows turn a darker shade of purple as you leave the graveyard of the forgotten dead for the familiar confines of your ship.`
				decline

mission "Tavish and Kormak 1"
	name "Tavish and Kormak"
	description "A pair of academics named Tavish and Kormak claim they are about to make a monumental discovery on <origin>. Take them to <destination> to pick up some mining tools. They can't pay you."
	source "New India"
	destination "Clark"
	passengers 2
	to offer
		random < 60
	on offer
		conversation
			`You make your way through the makeshift spaceport, breathing in the thick humid air and feeling the relentless glare of New India's twin suns. The muddy streets are filled with a constant bustling stream of humans, animals, and vehicles fighting either to unload the raw materials of this world, or to load up on the precious goods needed to tame it.`
			`	There's a small uncrowded cafe and bakery slightly off the beaten path. It seems like a nice place to get some respite from the crowds and the sun.`
			choice
				`	(Take a seat in the cafe.)`
					goto seat
				`	(Head back to your ship.)`

> 			`	You see no reason to stick around this rudimentary spaceport so you make your way back to your ship.`

Author: If you want to remove it, I am okay with that.

				decline

> 

Reviewer 2: ```suggestion

```
Merely adds in an empty line I removed by mistake.

				flee
			
			label yep
			`	The two explorers give each other a knowing nod, clearly pleased they could depend on you. They then begin the process of loading up your ship. Along with their weathered equipment they have multiple boxes filled with exotic xeno artifacts.`
			`	"Can't wait for the Society to see what we found!" Tavish beams as your ship is filled with their findings.`
			`	The final artifact is a huge lump of black and twisted alien wreckage that must weigh over 50 tons. It is being carried carefully by an army of strained robotic dollies that inch toward your vessel.`
			`	Tavish grins as he watches it get loaded. "The main ship itself was bricked. Old and mostly destroyed before it even hit the planet. But that piece broke off and was in slightly better shape. It's a little too big to show off to our fellows, but I do believe it will be worth a fair bit of credits to right patron. We can talk about that more in Alexandria."`
			`	With that everything and everyone is on board.`
			`	"Hup! Hup! To Alexandria!" Tavish shouts with obvious excitement.`
			`	Kormak cracks his knuckles and grins, "Finally."`

> 				flee

Reviewer 2: ```suggestion
				launch
```
I typed the wrong word in my previous suggestion.

			`	"At least that's what I believe," he says more to himself than to you.`
				decline

mission "Tavish and Kormak 1"
	name "Tavish and Kormak"
	description "A pair of academics named Tavish and Kormak claim they are about to make a monumental discovery on <origin>. Take them to <destination> to pick up some mining tools. They can't pay you."
	source "New India"
	destination "Clark"
	passengers 2
	to offer
		random < 60
	on offer
		conversation
			`You make your way through the makeshift spaceport, breathing in the thick humid air and feeling the relentless glare of New India's twin suns. The muddy streets are filled with a constant bustling stream of humans, animals, and vehicles fighting either to unload the raw materials of this world, or to load up on the precious goods needed to tame it.`
			`	There's a small uncrowded cafe and bakery slightly off the beaten path. It seems like a nice place to get some respite from the crowds and the sun.`
			choice
				`	(Take a seat in the cafe.)`
					goto seat
				`	(Head back to your ship.)`

			`	You see no reason to stick around this rudimentary spaceport, so you make your way back to your ship.`
				defer

> 

Reviewer 4: For whatever reason, these extra spaces sometimes cause issues ingame. I found once where the `on complete` didn't fire and once where the label didn't work:
![image](https://github.com/user-attachments/assets/5d70febf-6824-4b66-8c84-e72816dedd69)

They probably aren't breaking _everywhere_, but probably better to get rid of them just to be sure and to align with the structure of most other missions.
```suggestion
```

			label seat
			`	You take a seat near a window that does a surprisingly good job at muffling the noises beyond it. The cafe is quaint, nearly empty, and cool. Initially you're the only patron here, but after moment a pair of men enter.`
			choice
				`	(Check the menu.)`

> 

Reviewer 4: ```suggestion
```

			`	"... and you, my good fellow, have the look of someone who is just about to become of one our friends." Tavish flashes you a wide smile.`
			`	"You see, I and my fellow scholar are in desperate need of your eminent services. We are on the very cusp of a monumental discovery here in the untamed wilds of <origin>." Tavish leans in and gestures wildly with his hands. "The sort of discovery certain to rewrite the entire history of this sector! Why, science as we know it will never be the same."`
			`	"Never be the same," Kormak's gruff voice echos.`
			`	"Yes! Yes! But first we need a capable captain such as yourself to ferry us over to <destination> to pick up our equipment and bring us back here. A most trivial request for one of your talents, so trivial in fact you might even be inspired to waive your normal fee and do it pro bono; for science, that is."`
			`	"For science," Kormak echoes blandly.`
			choice
				`	"I'll do it, but not for free."`
				`	"Surely you have some money."`
				`	"Fine, I'll do it for free."`
					goto forscience
				`	"For science!"`
					goto forscience
				`	"Not interested."`
					goto notint

> 

Reviewer 4: ```suggestion
```

			`	"We are broke," Kormak helpfully adds.`
			`	"It is true," Tavish sighs. "Our circumstances are most unfortunate; we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccupation with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
			`	"We asked everyone," Kormak says.`
			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? We're sure our fortunes are about to change! So what do you say, my good fellow? Are you ready join us on our grand adventure and ferry us to the hardscrabble world of Clark?"`
			branch mayflower
				or
					has "first name: Mayflower"
					has "last name: Mayflower"

> 

Reviewer 4: ```suggestion
```

			choice
				`	"My name is <first> <last>, not Mayflower. You have no idea who I am, do you?"`
					goto embarrassing
				`	"I am not Mayflower. My name is Captain <last>."`
					goto embarrassing
				`	"You've mistaken me for someone else. I am Captain <last>, not Mayflower."`
					goto embarrassing

> 

Reviewer 4: ```suggestion
```

			label mayflower
			choice
				`	"Uh, yes?"`
					goto reluctantlyforscience
				`	"Sorry, I don't do 'pro bono' work for strangers."`
					goto notint

> 

Reviewer 4: ```suggestion
```

			`	"We asked everyone," Kormak repeats his comment from earlier.`
			`	"We must have gotten the flight manifest confused with yours and a different ship. I am truly apologetic for misleading you. To be frank, we have little idea who you are, Captain <last>."`
			`	"None," Kormak adds.`
			`	Tavish holds up a finger. "But, we can already tell you are a chap of quality, and the possessor of a keen curious mind. The sort of fellow we can put our absolute faith in. In fact, I see a lot of ourselves in you, truly we are kindred spirits."`
			`	"Like looking in mirror," Kormak says.`
			`	Tavish takes off his bowler hat. "So please, Captain <last>, now that you know our piteous plight, can you not find it in your heart - or better yet your inquirous mind - to ferry us without charge to Clark and back?"`
			choice
				`	"Fine, I'll do it for free."`
					goto reluctantlyforscience
				`	"For science!"`
					goto reluctantlyforscience
				`	"No, I only work for paying customers."`
				`	"No."`

> 

Reviewer 4: ```suggestion
```

			`	Tavish's bright eyes turn wet, and he drops his gaze to the table before you. Slowly, as if he's in actual pain, he places his bowler hat back atop his head. He then stands up, promptly followed by the lumbering Kormak.`
			`	"Very well. You're not the first captain to disdain science and cast us out into the streets, but I had truly hoped you'd be the first to see reason. I bid you good day."`
			`	As Tavish turns to leave, Kormak waivers for a just a moment. "Not like mirror," he finally says, then follows Tavish out.`
				decline

> 			

Reviewer 4: ```suggestion
```

			label notint

> 			`	Tavish's frowns, and he drops his gaze to the table before you. Slowly, as if he's in actual pain, he places his bowler hat back atop his head. He then stands up, promptly followed by the lumbering Kormak.`

Reviewer 4: ```suggestion
			`	Tavish frowns, and he drops his gaze to the table before you. Slowly, as if he's in actual pain, he places his bowler hat back atop his head. He then stands up, promptly followed by the lumbering Kormak.`
```

			`	"Very well. There will be other captains who recognize our proposal for the boon it is. I bid you good day."`
			`	As Tavish turns to leave, Kormak waivers for a just a moment. "Fool," he finally says, and then follows Tavish out.`
				decline

> 				

Reviewer 4: ```suggestion
```

			label reluctantlyforscience
			`	Tavish breaks into a wide grin and mock punches the air just above his head. "Huzzah, my good chap! Yes, that is the pioneering spirit!"`
			`	He abruptly turns to Kormak and they share a few congratulatory handshakes.`
			`	Once the handshaking concludes Tavish carefully rises from his seat, "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice
				`	"The <ship>."`
			`	"Right, of course, a ship worthy of a captain of your caliber, no doubt, no doubt." Tavish tips his hat to you, and then he and Kormak make their way out of the cafe.`
				accept

> 

Reviewer 4: ```suggestion
```

			`	"We are broke," Kormak helpfully adds.`
			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccupation with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
			`	"We asked everyone." Kormak says.`
			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`
			branch mayflower2
				or
					has "first name: Mayflower"
					has "last name: Mayflower"

> 

Reviewer 4: ```suggestion
```

			choice
				`	"Yes! To Clark! Although my name is not Mayflower, it's <last>.`
					goto embarrassment2
				`	"Fine, but I better not regret this. Also, my name is <last>, not Mayflower."`
					goto embarrassment2
				`	"Sure, whatever, but where did you get Mayflower from? I'm Captain <last>."`
					goto embarrassment2

> 

Reviewer 4: ```suggestion
```

			label mayflower2
choice
				`	"Yes! To Clark!`
					accept
				`	"Fine, but I better not regret this."`
					accept
				`	"Sure, whatever."`
					accept

> 

Reviewer 4: ```suggestion
```

			`	"Er, yes, about that..." there's a pause before Tavish, his face red, finally says, "We may have been a tad bit desperate, and have been asking every captain who lands here, regardless of reputation." `
			`	"We asked everyone," Kormak repeats his comment from earlier.`
			`	"I must have gotten the flight manifest confused with yours and a different ship. I am truly apologetic for misleading you. To be frank we have no idea who you are, Captain <last>."`
			`	"None," Kormak adds.`
			`	Tavish holds up a finger. "But, we can already tell you are a chap of quality, and the possesor of a keen curious mind. The sort of fellow we can put our absolute faith in. In fact, I see a lot of ourselves in you, truly we are kindred spirits."`
			`	"Like looking in mirror," Kormak says.`
			`	Tavish suddenly stands. "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice
				`	"The <ship>."`
			`	"Right, of course, a ship worthy of a captain of your caliber, no doubt, no doubt." Tavish tips his hat to you, and then he and Kormak make their way out of the cafe.`
				accept

> 

Reviewer 4: ```suggestion
```

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

> 

Reviewer 4: ```suggestion
```

			`	"Yes! Yes! But first we need a capable captain such as yourself to ferry us over to <destination> to pick up our equipment and bring us back here. A most trivial request for one of your talents, so trivial in fact you might even be inspired to waive your normal fee and do it pro bono; for science that is."`
			`	"For science," Kormak echoes blandly.`
			choice
				`	"I'll do it, but not for free."`
				`	"Surely you have some money."`
					goto cashmoney
				`	"Fine, I'll do it for free."`
					goto forscience
				`	"For science!"`
					goto forscience
				`	"Not interested."`
					goto notint

			label cashmoney
			`	Tavish glances at Kormak and then back to you. There's a hint of sweat on his brow as he says, "Ahem, my esteemed colleague and I have run into to some trouble, of the financial variety. You see, we uh, well, already paid another very charming captain to pick up our equipment on Clark, and they never returned. We then paid another captain to take us there, but when we went to meet them at their ship we discovered the most unwelcome news that there was no ship at all, and they were nowhere to be found."`
			`	"We are broke," Kormak helpfully adds.`

> 			`	"It is true," Tavish sighs. "Our circumstances are most unfortunate; we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`

Author: Preoccupation is the intended word here.

> 			`	"It is true," Tavish sighs. "Our circumstances are most unfortunate; we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`

Reviewer 2: ```suggestion
			`	"It is true," Tavish sighs. "Our circumstances are most unfortunate; we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccupation with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
```

			`	He abruptly turns to Kormak and they share a few congratulatory handshakes.`
			`	Once the handshaking concludes Tavish carefully rises from his seat, "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice
				`	"The <ship>."`
			`	"Right, of course, a ship worthy of a captain of your caliber, no doubt, no doubt." Tavish tips his hat to you, and then he and Kormak make their way out of the cafe.`
				accept

			label forscience
			`	Tavish breaks into a wide grin and mock punches the air just above his head, "Huzzah, my good chap! Yes, that is the pioneering spirit! Just the news we needed to hear, for I must admit to you, that my esteemed colleague and I have run into to some trouble of the financial variety. We've already paid another very charming captain to pick up our equipment on Clark, and they never returned. We then paid another captain to take us there, but when we went to meet them at their ship we discovered the most unwelcome news that they in fact were not its owner."`
			`	"We are broke," Kormak helpfully adds.`

> 			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`

Author: again, preoccupation is the intended word.

> 			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccuption with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`

Reviewer 2: ```suggestion
			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccupation with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
```

			`	"We asked everyone." Kormak says.`

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible and we shall commence with our grand adventure to the hardscabble world of Clark!"`

Author: we is the wrong word here... maybe _then_? When you are ready... _then_ we shall commence?

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible and we shall commence with our grand adventure to the hardscabble world of Clark!"`

Reviewer 2: updated

			choice
				`	"Fine, but first you need to answer some questions."`
					goto questions
				`	"Nope. I don't know what this is, but I am out."`

			label quit
			`	Tavish frowns and sighs, but says nothing. He looks at Kormak and then begins to wander away.`
			`	Kormak glares at you and then says, "Could have had fun."`
			`	The large man turns and follows Tavish back into Clark's wild spaceport.`
				decline
			
			label questions
			`	"Of course." Tavish nods emphatically, "We assuredly are an open book."`
			`	Kormak delivers a dissenting grunt.`
			`	"Uh, mostly open." Tavish quickly amends.`

			label questions2
			choice
				`	"Just who exactly are you?"`
					goto who1
					to display
						not "who1"
				`	"What sort of science are you doing on New India?"`
					goto india1
					to display
						not "india1"
				`	"What is this equipment you need so badly?"`
					goto equipment1
					to display
						not "equipment1"
				`	"I have no more questions. But from now on I'm in charge. Now, let's go to Deadman's Cove."`

> 					goto end

Author: How about, "I am done. For now on in' in charge."

> 					goto end

Reviewer 2: updated

			`	Tavish thanks the man and promises him he'll be seeing his first returns in the near future, and with that the three of you head out of the bar.`

			label story
			choice
				`	"Tavish, what's your story?"`
				`	"Tavish that was amazing! Where did you learn to do that?"`
				`	"Tell me the truth, Tavish. Are you conning me too?"`
			`	As you walk toward your ship Tavish shrugs, "Not everything I say is a fabrication. Why, I really am from Chiron, and was a highly placed scientist and researcher there. But the bureaucracy there was cut-throat, and anyone with a renegade idea was liable to be crushed by it."`
			`	He shakes his head sadly, "In order to survive in that environment I had to become very good at working with those around me. Make them believe we were on the same side, believed the same things, had the very same lofty goals. I got quite good at it."`
			`	His voice becomes somber, "Perhaps too good. I started to forget where the genunine Tavish began, and the counterfeit Tavish ended. So, I stopped hiding who I was and made my interests and studies known to all!"`

> 			`	A wry grin spreads across his face, "And was promptly thrown out! So, now I am here with you and Kormak following my passion, on the forefront of discovery and adventure! And I am not too disconcerted that my old talents have some use."`

Author: One of these days I think we should talk about how to tell if its going to be a comma or a period after a statement before dialog, so I can learn it and you can stop fixing all these mistakes for me

> 			`	A wry grin spreads across his face, "And was promptly thrown out! So, now I am here with you and Kormak following my passion, on the forefront of discovery and adventure! And I am not too disconcerted that my old talents have some use."`

Reviewer 3: It's always a full stop, unless the dialogue follows a verb denoting speech.

> He shouts, "what's wind chill factor?"

> He went up to the podium. "What's wind chill factor?"

> 			`	A wry grin spreads across his face, "And was promptly thrown out! So, now I am here with you and Kormak following my passion, on the forefront of discovery and adventure! And I am not too disconcerted that my old talents have some use."`

Author: I'll try to remember this.

	source "Hephaestus"
	destination "Chiron"
	passengers 2
	to offer
		has "Tavish and Kormak 3: done"
	on offer
		conversation
			`You get the feeling the sun never sets on Hephaestus as you navigate through droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of the human expansion, and not Earth.`
			`	Tavish leads you down one narrow corridor filled with endless ships. You don't see any labels or signage that make sense, but somehow Tavish plows on with a quick step and unshakable confidence.`
			`	Eventually he leads you into a large administrative complex. After four flights of stairs, three different elevators, and one very small trolley the three of you arrive at a bland corporate office with the words, "Repository of Confiscated Goods."`
			`	There are no doors, and a single window that looks like it was made of reinforced glass. The tired looking agent on the other side of the glass hits a button and his voices buzzes over to you, "What can I help you with?"`
			choice

> 				`	"We're looking for stolen scientific research equipment that was recently confiscated aboard the Broken Record."`

Author: done.

			`	Tavish pauses. "Although, don't repeat that story to Kormak. It upsets him. I think the Ice-Witches of Nifel may be distant cousins of his."`
				goto options
			
			label henry
			`	"Dr. Henry Forsyth saved millions, maybe billions of lives, by ignoring the academic consesus of the day, and putting his life on the line by leading a daring expedition into the frozen wastes of a desolate world. There he discovered the barbaric Ice-Witches of Nifel, and seduced their wild and beautiful leader, the Ornery-dress. She would unwittingly lead him to the cure of the Rigellian Plague, and secure his position as a preeminent scholar, exlorer and hero of his time."`
			`	He seems to inflate as he speaks, "That's the legacy of SOGEE, of the Society of Gentlemanly Exemplary Explorers, and I may not ever live up to Dr. Forsyth, but I still believe in the spirit of his actions and am committed to following his footsteps."`
			`	Tavish pauses. "Although, don't repeat that story to Kormak. It upsets him. I think the Ice-Witches of Nifel may be distant cousins of his."`
				goto options

> 

Author: I'd like to disagree with removing it as its own mission. This gives the player and Tavish time to mull over what has happened, it also breaks things up a bit by having this occur elsewhere - rather than forcing the player to read everything at once. And I think demonstrating that on your ship you can have a conversation with someone as you travel is a nice touch.

> 

Reviewer 2: My suggestion is moving it to do the exact same thing, except for in the same mission definition to save space.

> 

Author: oh, okay.

mission "Tavish and Kormak 5"
	name "New India at last"
	description "Transport Tavish and Kormak to <planet> to resume their expedition."
	source "Chiron"
	destination "New India"
	passengers 2

> 	cargo "Scientific Research Equipment" 5

Author: Done.

			`	Over to the side Tavish has also finished his work and is making a campfire.`
			`	"Come," Kormak says.`

			label next
			choice
				`	(Take a seat around the campfire.)`

			`	As the night takes hold, strange alien noises can be heard coming from the jungle. Tavish continues to tend the fire as Kormak starts roasting rations to pass around.`
			`	Tavish takes a seat and a bite of the roasted food that Kormak has put into his hand.`
			`	"Good expedition," Kormak says beneath the emerging stars.`
			`	Tavish nods, "Certainly more than any of us bargained for, but tomorrow the real work begins and who knows what wonders we will discover in that ancient crashed ship."`
			`	"Yes," Kormak says between bites. "Where I from songs would be sung of our deeds."`
			choice
				`	"Kormak, where are you from?"`
				`	(Say nothing.)`
					goto notathing

			`	Kormak stops chewing and looks hard at you for a long moment before finally says, "It is forbidden."`
			`	"Ah, about that," Tavish jumps in, "the thing is the Deep is more complex than most people realize. It is a society with three distinct layers."`

> 			`	Tavish mimes peeling a fruit, "The top layer is the one everyone knows about. It's clean, fair, democratic, and technologically advanced. It's the Deep you hear about in holo-dramas, and see talked about on the news. Most of the Deep is a part of this layer.`

Reviewer 2: ```suggestion
			`	Tavish mimes peeling a fruit. "The top layer is the one everyone knows about. It's clean, fair, democratic, and technologically advanced. It's the Deep you hear about in holo-dramas, and see talked about on the news. Most of the Deep is a part of this layer.`

```

			`	Kormak frowns, "Lies."`
			`	"Or, and this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`
			`	"Not elves," Kormak interrupts with a hint of annoyance, "the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`
			`	Kormak sighs and turns back to you, "If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf. Know that is my home."`
			`	"Enough about past. Only present matters." He goes back to eating.`
			`	Tavish laughs, "Of course. Let us discourse about less weighty matters."`

			label notathing
			`	You talk a little of unimportant and unremarkable things beneath the glittering twilight stars in the midst of an untamed alien jungle...`
			`	And then you head to bed.`
			choice
				`	(Sleep.)`

			`	You are awoken by something warm and wet slowly sliding up your leg. Your tent has been perforated, and you can tell the blazing ring of fire is off.`
			choice
				`	(Kick whatever is on your leg.)`

> 			`	With a few stiff kicks you force the grasping appendage to lets go of your leg and slithers back out the tent hole. You follow after it to see the entire campground is writhing in tenticle like vines.`

Reviewer 2: ```suggestion
			`	With a few stiff kicks you force the grasping appendage to let go of your leg and slither back out of the tent hole. You follow after it to see the entire campground is writhing in tentacle-like vines.`
```

			label everyone
			`	Everyone is now up and attacking the vines. Kormak begins laughing and starts singing in his bloodlust:`
			``
			`	Axe-Edge, Bright-Pledge, Breaker of Blight's Dredge,`
			`	Ash-born, Eagle-torn, Slayer of Earth's forlorn.`
			`	Break before my battle axe, Flee before my thunder.`
			`	Die beneath a stranger's sky, and feed my clan's hunger.`
			``
			`	After a while nothing moves. It seems all the alien vines have retreated back behind the tree line. Belatedly you realize they weren't terribly formidable. The vines were slow, weak, and soft.`
			choice
				`	"That's it? How did those things manage to kill Gerald?"`
				`	"Truly we've won a glorious victory today."`
					goto victory
				`	"I am going back to bed.`
					goto wakeup

			`	Kormak and Tavish look at each other and then finally Tavish says, "He really was quite slow."`
			`	Kormak nods. "Very slow."`
			`	"And he may have been aggressively afraid," Tavish adds.`
			`	"Rolled up into ball," Kormak confirms. "Very sad. I kill trees in his memory."`
			`	Tavish solemnly agrees and then tinkers with the ring of fire for a moment. "Ah, here's the problem," he says, and switches it back on.`

> 			`	"We'll be completely safe." He says and heads back into his tent.`

Reviewer 2: ```suggestion
			`	"We'll be completely safe," he says, and heads back into his tent.`
```

			choice
				`	(Go back to sleep.)`
					goto wakeup

			label victory
			`	Both Kormak and Tavish cheer.`

> 			`	"You handled yourself quite well, Captain <last>." Tavish says. "Much better than Gerald."`

Reviewer 2: ```suggestion
			`	"You handled yourself quite well, Captain <last>," Tavish says. "Much better than Gerald."`
```

			`	Kormal gives you a solemn nod, "Very slow. Just rolled into ball. Said 'take me,' and gone."`
			`	Tavish nods, "We'll dedicate the paper we write on this expedition to his memory. Such are the risks we explorers must face."`
			`	"Risks." Kormak agrees.`
			`	Tavish tinkers with the ring of fire for a moment. "Ah, here's the problem," he says, and switches it back on.`
			`	"We'll be completely safe." He heads back into his tent.`
			choice
				`	(Go back to sleep.)`
			
			label wakeup

> 			`	The morning comes without further incident. You eat breakfast, trade a few more harrowing tales about your adeventures in the night and prepare to leave.`

Reviewer 2: ```suggestion
			`	The morning comes without further incident. You eat breakfast, trade a few more harrowing tales about your adventures in the night, and prepare to leave.`
```

			`	"Appreciate it," Kormak echoes.`
			`	Tavish looks a little apprehensive, "Without your aid our predicament will be most perilous. Between the wreckage and our equipment I hypothesize we will have at least 90 tons of cargo to transplant. However, we are still low on funds. It may prove challenging to persuade another ship to transport us pro bono via radio. So please do come back in two months."`
			choice
				`	"I'll come back. Don't worry."`
				`	"We'll see. No promises."`
				`	"I do what I want."`
			`	Tavish nods, "In that case we've quite a bit of work to do, Captain. Take care."`
			`	He starts working on prepping the dig site, while Kormak escorts you back to your ship.`
			`	Kormak takes your hand in his. "May the stones guide you," he says, and then disappears back into the jungle.`

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

event "Tavish and Kormak Expedition"

> mission "Tavish and Kormak 7"

Reviewer 2: ```suggestion
mission "Tavish and Kormak 7"
	landing
```

	name "Alexandria awaits"
	description "Transport Tavish and Kormak and their cargo to <planet>."
	to offer
		has "event: Tavish and Kormak Expedition"
	source "New India"
	destination "Alexandria"
	passengers 2
	cargo "Scientific Research Equipment" 90
	on offer
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice
				`	(Land on Ashoka and pick up Travis and Kormak.)`
					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka
			`	Tavish and Kormak are waiting with all their equipment near the stony outcrop where you landed before. They seem more than a little relieved to see you.`

> 			`	"Knew we could count on you." Tavish happily says as he oversees the loading up of the equipment.`
> 			`	"Honorable." Kormak adds.`
> 			`	Tavish gives you a serious look, "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`
> 			`	"We have no money." Kormak helpfully adds.`

Reviewer 2: ```suggestion
			`	"Knew we could count on you," Tavish happily says as he oversees the loading up of the equipment.`
			`	"Honorable," Kormak adds.`
			`	Tavish gives you a serious look. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`
			`	"We have no money," Kormak helpfully adds.`
```

			choice
				`	"Of course! To Alexandria we go!"`
					goto yep
				`	"Fine! But you owe me."`
					goto yep
				`	"No. I am done giving out free rides."`
			`	With surprised, crestfallen faces, the two explorers just stare at you. They are both at a loss for words as they watch you abandon them to their fates on the dark side of New India.`

> 				decline

Reviewer 2: This currently means you get access to the facilities and everything. I believe `flee` would work better, because it launches you.
```suggestion
				flee
```

			
			label yep

> 			`	The two explorers give each other a knowing nod. Clearly pleased they could depend on you, and then begin the process of loading up your ship.`
> 			`	Along with their weathered equipment they have multiple boxes filled with exotic xeno artifacts.`

Reviewer 2: ```suggestion
			`	The two explorers give each other a knowing nod, clearly pleased they could depend on you. They then begin the process of loading up your ship. Along with their weathered equipment they have multiple boxes filled with exotic xeno artifacts.`
```

			`	"Can't wait for the Society to see what we found!" Tavish beams as your ship is filled with their findings.`
			`	The final artifact is a huge lump of black and twisted alien wreckage that must weigh over 50 tons. It is being carried carefully by an army of strained robotic dollies that inch toward your vessel.`
			`	Tavish grins as he watches it get loaded. "The main ship itself was bricked. Old and mostly destroyed before it even hit the planet. But that piece broke off and was in slightly better shape. It's a little too big to show off to our fellows, but I do believe it will be worth a fair bit of credits to right patron. We can talk about that more in Alexandria."`
			`	With that everything and everyone is on board.`
			`	"Hup! Hup! To Alexandria!" Tavish shouts with obvious excitement.`
			`	Kormak cracks his knuckles and grins, "Finally."`

> 				accept

Reviewer 2: ```suggestion
				flee
```
Same reasoning.

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
	on complete
		outfit "Salvaged Artifact" 1
		conversation

> 			`	Kormak and Tavish have changed back into their gentlemanly attire and eagerly oversee the unloading of their instruments and artifacts into Alexandria. A handful of similarly overdressed gawkers have come to watch the process. A few of them are eager to congratulate Tavish and Kormak.`

Reviewer 2: ```suggestion
			`Kormak and Tavish have changed back into their gentlemanly attire and eagerly oversee the unloading of their instruments and artifacts into Alexandria. A handful of similarly overdressed gawkers have come to watch the process. A few of them are eager to congratulate Tavish and Kormak.`
```

			`	Tavish does not unload the large piece of alien equipment off your ship, and instead comes to speak with you.`

> 			`	"This is where we part ways, Captain <last>." Tavish says with a hint of sadness. "It was truly an adventure. Perhaps one day our paths will cross again."`

Reviewer 2: ```suggestion
			`	"This is where we part ways, Captain <last>," Tavish says with a hint of sadness. "It was truly an adventure. Perhaps one day our paths will cross again."`
```

				defer
			label enterstore
			`	The dingy run-down shop is filled with clutter, piles of unrecognizable technology, and miles of tangled cords.`
			`	"Don't mind the mess," a gravelly voice interjects from somewhere deeper in the store.`
			`	A few moments later, a well-built, graying man painfully hobbles into the front. "Name's Weland, what can I do for you?"`
			choice
				`	(Tell him about the Salvaged Artifact.)`
					goto AA
				`	(Ask him what he does.)`
			`	"Bit of this, bit of that." Weland shrugs. "Gotta knack for figurin' things out, and the people, well, they bring me things they need some figurin' on. Sometimes I get'em workin'. Sometimes I buy 'em. Sometimes I even make somethin' new.`
			`	"So, you need me to figure somethin' out? Or you lookin' to buy? I've got cords, lots of cords, cheap even. Colonial era cords, alien cords, cords from the Deep, cords that only work on two Paradise worlds. I even got a cord that'll charge up a thousand year old cell phone. I mean, I have other stuff too, probably better, but I really need to offload some of these cords."`
			choice
				`	(Tell him about the Salvaged Artifact.)`
					goto AA

> 				`	(Ask him about Weapons.)`

Reviewer 2: ```suggestion
				`	(Ask him about weapons.)`
```

			`	Weland strokes his beard for a moment, and then says, "Listen here. Maybe I can help. I'll need to take a look at it. Stories are great and all, but I gotta put eyeball to metal to know if what you're talking about is wortha damn, and then even if it is valuable and if there is anything I can do about it."`
			choice
				`	(Agree to leave the artifact with Weland.)`
					goto agreetoweland
				`	(Refuse to leave the artifact with Weland.)`
			`	Weland holds up his hands. "Alright, now, just hold up, you really want to keep dragging around some useless cargo from port to port? I'm not going nowhere." Weland lifts up one of his mangled feet, there's some sort of chain directly embedded into his ruined achilles heel. "Not as if I could run, even if I wanted to."`
			`	"So, you sure you want to pass up on this? That you can't give me a few days to look at your artifact?"`
			choice
				`	(Agree to leave the artifact with Weland.)`
					goto agreetoweland
				`	(Refuse to leave the artifact with Weland.)`
			`	Weland shrugs. "If that's how you feel. Well, before you go, any chance I could offer you some premium cords at crazy discount prices?"`

> 			`	Weland then launches into another sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should ever need a cord, and promptly leave.`

Reviewer 2: ```suggestion
			`	Weland then launches into another sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should you ever need a cord, and promptly leave.`
```

			`	He gives you a pained smile. "I know what you're thinking. Be a hero, help the old man, some such nonsense. This isn't that story, and you aren't that person. So, let it rest. These are my chains, and this is my problem, and one way, or another, I'll deal with it. So, is your curiosity sated? Can we return to the matter at hand?"`
				goto questions
			label dragon
			`	"Yep, Dragons. Well, not the kind from Earth stories, or, at least, Earth, probably. We find 'em in the Rim, the South, the Dirt Belt, bits and pieces of them, that is. Reckon they must have lived here more than a hundred thousand years ago. Most their tech is in ruins, broken, dilapidated, hard to piece much of it together. Suffice to say, they were big brutes, they had big space ships, and they've been dead for a helluva long time."`
			`	He looks over at your artifact, "And you, well, you found somethin' of theirs."`
				goto questions
			label next
			`	"Getting somethin' like this up, and running, well, it's not quick, it's not easy, and it won't be cheap."`

> 			`	He holds his hand up. "'Fore you say anythin', what I did was run a preliminary check. Had to make sure it wasn't trash, or useless, and waste both our time. You'd be surprised the nonsense folk drag in here. I still don't know it functions, let alone, how to integrate it with your ship. But, with time, I will."`

Reviewer 2: ```suggestion
			`	He holds his hand up. "'Fore you say anythin', what I did was run a preliminary check. Had to make sure it wasn't trash, or useless, and waste both our time. You'd be surprised the nonsense folk drag in here. I still don't know it functions, let alone how to integrate it with your ship. But, with time, I will.`
```

			`	"So, here's what I can offer you. Give me 6 months. You might think that's long, but you won't find anybody who can do it faster. I got a few ideas, seen a few things, and am confident I can unravel this here little mystery. Now, my fee will be a million credits a month, which should come out to 6 million credits total. That may sound steep, but you will only have to pay if I get it workin'. So, whatd'ya say? You in?"`
			choice
				`	(Agree to his terms.)`
					goto agree
				`	(Refuse his terms.)`
			`	Weland frowns. "Without me, this'll just be one heavy good luck charm. And, I will admit, I am plumb excited to get under the hood. I like me a challenge. Sure you won't reconsider?"`
			choice
				`	(Agree to his terms.)`
					goto agree
				`	(Refuse his terms.)`
			`	"Well, can't say as I am not disappointed to hear you say that. But, it is your decision. I'll have the artifact moved back to your ship. Now, before you go, let me tell you about a few real nice cords I got for sale."`
			action
				outfit "Salvaged Artifact" 1

> 			`	Weland then launches into sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should ever need a cord, and promptly leave.`
> 			`	Weland is a man of his word, and your Salvaged Artifact is returned to your ship.`

Reviewer 2: ```suggestion
			`	Weland then launches into sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should you ever need a cord, and promptly leave.`
			`	Weland is a man of his word, and your Salvaged Artifact is soon returned to your ship.`
```

				decline
			label agree
			action

> 				event "welandwait2"

Reviewer 2: ```suggestion
				event "welandwait2" 160 180
```

> 			`	Weland beams. "That's the spirit, now. Figurin' things out, well, that's what it means to be human, isn't it? Now, as I said, I reckon it'll take me near about 6 months to get this beauty figured out. I realize, being a fancy space hero an' all, you probably will need a little reminder, so when I am done I'll send you a message. As I promised, if I can't get it to work, you don't have to pay me a penny. But, well, wouldn't be betting on that happening, I haven't yet met a machine I couldn't crack."`

Reviewer 2: ```suggestion
			`	Weland beams. "That's the spirit, now. Figurin' things out, well, that's what it means to be human, isn't it? Now, as I said, I reckon it'll take me near about 6 months to get this beauty figured out. I realize, being a fancy space hero an' all, you probably will need a little reminder, so when I am done I'll send you a message. As I promised, if I can't get it to work, you don't have to pay me a penny. But, well, wouldn't be betting on that happening; I haven't yet met a machine I couldn't crack."`
```

			`	The large man turns from you and finishes chopping down the last wailing tree.`
			`	With the area around the designated campground clear he faces you and gives you the smallest nod, "Trust."`
			`	Over to the side Tavish has also finished his work and is making a campfire.`
			`	"Come," Kormak says.`

			label next
			choice
				`	(Take a seat around the campfire.)`
				`	(Join Tavish and Kormak.)`

			`	As the night takes hold, strange alien noises can be heard coming from the jungle. Tavish continues to tend the fire as Kormak starts roasting rations to pass around.`
			`	Tavish takes a seat and a bite of the roasted food that Kormak has put into his hand.`
			`	"Good expedition," Kormak says beneath the emerging stars.`
			`	Tavish nods, "Certainly more than any of us bargained for, but tomorrow the real work begins and who knows what wonders we will discover in that ancient crashed ship."`
			`	"Yes," Kormak says between bites. "Where I from songs would be sung of our deeds."`
			choice
				`	"Kormak, where are you from?"`
				`	(Say nothing.)`
					goto notathing

> 			`	Kormak stops chewing and looks hard at you for a long moment before finally says, "It is forbidden."`

Author: Added your suggestion, and another potential one for if the player has finished the main story.  Should help to break it up.

			`	"And then," Tavish continues, "there's the middle layer. It is smaller and made up of people who practice what they call the old ways. Those who practice stone-funerals, or who pay some respect to the scattered God-Stones. It's people who still see themselves as the descendants of those first brave pilgrims that colonized the Deep. Like Hroar's Haulers."`
			`	Tavish pinches his fingers together like he's holding his invisible fruit's core, "Finally, there's the lowest layer. Few are aware of it, and I think it may be an embarrassment to leaders of the Deep. It's made up of people like Kormak, or the Ice-Witches of Nifel."`
			`	"The Ui Skathi," Kormak says. "Granddaughters of Skathi."`
			`	"Was that what they called themselves?" Tavish shrugs. "Kormak's always a little sensitive about this. Anyways, they don't really make sense. How did they get to the Deep? Where did they come from? There's two prevailing theories. First, that there was a lost generational fleet that landed in the Deep earlier than anyone realized, that something went wrong, and that they degenerated in primitives."`
			`	Kormak frowns, "Lies."`

> 			`	"Or, and this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`

Author: Done!

			`	"Appreciate it," Kormak echoes.`
			`	Tavish looks a little apprehensive, "Without your aid our predicament will be most perilous. Between the wreckage and our equipment I hypothesize we will have at least 90 tons of cargo to transplant. However, we are still low on funds. It may prove challenging to persuade another ship to transport us pro bono via radio. So please do come back in two months."`
			choice
				`	"I'll come back. Don't worry."`
				`	"We'll see. No promises."`
				`	"I do what I want."`
			`	Tavish nods, "In that case we've quite a bit of work to do, Captain. Take care."`
			`	He starts working on prepping the dig site, while Kormak escorts you back to your ship.`
			`	Kormak takes your hand in his. "May the stones guide you," he says, and then disappears back into the jungle.`

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

event "Tavish and Kormak Expedition"

> mission "Tavish and Kormak 7"

Author: Oh man. I should make it so if you're too late they're dead.

> mission "Tavish and Kormak 7"

Author: Alright. I acted upon your excellent suggestion. There are now three possible interactions depending on how late you arrive to pick them up.

			`	Tavish does not unload the large piece of alien equipment off your ship, and instead comes to speak with you.`
			`	"This is where we part ways, Captain <last>," Tavish says with a hint of sadness. "It was truly an adventure. Perhaps one day our paths will cross again."`
			`	Tavish sniffles for a moment. "Apologies, allergies. Anyways, old chap, I just wanted you to know that Kormak and I, we pay our debts. And though we have no money to offer you, we are instead bequeathing to you the large artifact we found. We scanned it pretty thoroughly and we don't believe it has additional scientific value. However, we think a collector would pay well for it. Quite well."`
			`	He leans a little closer to you, "The best place to look for that sort of commerce is Wayfarer."`
			choice
				`	"Thanks, Tavish. It's been grand."`
				`	"This better be worth it."`
			`	A tear edges its way down Tavish's nose. "Thanks again, Captain. And farewell."`

> 			`	Tavish turns and leaves you to rejoin Kormak and the excited hovering members of the Society of Gentlemanly Exemplary Exploration.`

Reviewer 4: Similar to above, would be good to have a log for the player to reference that the best place to sell the artifact is on Wayfarer.

Author: Added a log entry.

			`	"So, you sure you want to pass up on this? That you can't give me a few days to look at your artifact?"`
			choice
				`	(Agree to leave the artifact with Weland.)`
					goto agreetoweland
				`	(Refuse to leave the artifact with Weland.)`
			`	Weland shrugs. "If that's how you feel. Well, before you go, any chance I could offer you some premium cords at crazy discount prices?"`
			`	Weland then launches into another sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should you ever need a cord, and promptly leave.`
				decline
			label agreetoweland
			action
				event "welandwait1" 6
			`	The old man grins brightly. "Good. Good. I 'spect it'll take me about a week to get my bearings on this artifact of yours. I'll also be needing your ship's schematics, it'll help me figure out how it can be installed, assumin' you want that."`
			`	With the deal completed, you head back to the spaceport. You send the pertinent information about your ship to Weland, and have the outfit transferred to him as well. Hopefully, he and your artifact are still here when you return after a week.`

> 			decline

Reviewer 4: ![image](https://github.com/user-attachments/assets/d4bca83e-25c5-4e83-ab24-a5337646f6a4)

```suggestion
```

event "welandwait1"

mission "Weland 2"
	invisible
	source "Wayfarer"
	landing
	to offer
		has "event: welandwait1"
	on offer
		conversation
			`Weland should be done with his preliminary study of your artifact by now. You head back to the bazaar to check on him.`
			`	The door to his seedy-looking store is unlocked, but there is no one among the piles of cords, dust, and junk. The place is empty.`
			`	You search his cramped shop for clues, and behind a sagging dusty bookshelf you find a passage that leads into an adjacent building.`
			choice
				`	(Follow the passage nextdoor.)`
					goto nextdoor
				`	(Leave.)`
			`	You back away from the passage. Maybe you can find him another time. You leave his odd shop and head back to your ship.`
				defer

			label nextdoor
			`	The building, which on the outside looks like an abandoned warehouse, is in fact a sprawling laboratory.`
			`	Exotic technology and strange artifacts fill the room and at its center is Weland, leaning over a sophisticated workbench.`
			`	"Well, look what the cat dragged in. 'Bout time you showed up." Weland flicks his red-tinted goggles off his head and slowly stands.`

> 			`	Weland nods toward a mass of technology, that you realize is incasing your artifact, and limps over to it. As he's walking you notice thick surgically connected chains dragging behind his feet.`

Reviewer 4: ```suggestion
			`	Weland nods toward a mass of technology that you realize is encasing your artifact, and limps over to it. As he's walking you notice thick surgically connected chains dragging behind his feet.`
```

			`	"Sorry man, good luck with your science stuff or whatever. But that's all I know so I'm out of here."`
			`	Hans waits until you transfer the money over to him, and then quickly retreats deeper into his ship.`
			`	Tavish watches him go. "I dare say he didn't deserve additional recompense after what he did to us."`
			`	Kormak grunts in agreement.`
			label end

	on visit
		dialog `You haven't found the Broken Record yet.`
	on complete
		conversation
			`Your ship lands in the heart of the continent-consuming megatropolis Polyphron, the "only city" on Hephaestus.`
			`	Finding your way around here could well be impossible. Thankfully Tavish says he knows the area where recently impounded ships are housed.`

> 			`	He and Kormak head toward the spaceport.`

Author: I am the fence on this. I feel like it's clearly implied in the original, whereas the added sentence feels almost heavy handed. I'd appreciate it if anyone else could chime in and say which version they prefer.

mission "Tavish and Kormak 4"
	name "A journey to Chiron"
	description "Transport Tavish and Kormak to <planet> to look for a man named Doctor-Professor Dylan Hullen-Freeze."
	source "Hephaestus"
	destination "Chiron"
	passengers 2
	to offer
		has "Tavish and Kormak 3: done"
	on offer
		conversation

> 			`You get the feeling the sun never sets on Hephaestus as you navigate through droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of the human expansion, as opposed to Earth.`

Author: lol I am pretty sure Giraffe just had me change it from "_rather than_" to "_as opposed to_". His rational being that you could read it as the place you are as being Earth.

> 			`You get the feeling the sun never sets on Hephaestus as you navigate through droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of the human expansion, as opposed to Earth.`

Reviewer 4: lol, my thought reading it was that "as opposed to" implies that Earth and Hephaestus are opposites, but if it was changed to that it's probably fine. Just delete the extra "the" and this one's good then.

			`	They pause, as if for dramatic effect, and then continue, "Chiron Campus Oxford-Harvard Research Office. Department of Oddities. Doctor-Professor Dylan Hullen-Freeze. That is all the information I have. Good day."`
			`	The window suddenly goes opaque.`
			`	Tavish swallows. "This is most disconcerting."`
			`	Without looking at you or Kormak, he heads toward the exit.`
			`	"Bad history," Kormak says, before following Tavish out.`
			`	It seems Chiron is your next destination.`
				accept

			label charges
			`	"Of course. By allowing your equipment to be stolen you are encouraged criminal elements in the Syndicate. Now, is there anything else I can help you with?"`
			choice
				`	"Is there anyway we can collect our stolen equipment?"`
				`	"We just want our stuff back."`
					goto collect

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish were on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

> 	on enter

Author: Not a duplicate. There used to be green text explaining the use of the second on enter, but Giraffe re-sorted it all. 

The second on Enter helps the mission happen in space, and not on world.

> 	on enter

Reviewer 4: Huh, weird. Feel free to resolve this comment then.

			`	It's obvious Tavish is uncomfortable here, but he does his best not to show it. He informs you that you'll need to get to the commuter rail deeper in the New Port spaceport to meet Dr. Hullen-Freeze.`

mission "Tavish and Kormak 5"
	name "New India at last"
	description "Transport Tavish and Kormak to <planet> to resume their expedition."
	source "Chiron"
	destination "New India"
	passengers 2
	cargo "Scientific Research Equipment" 5
	blocked "You have reached <origin>, but you need <capacity> in order to take on the next mission. Return here when you have the required space free."
	to offer
		has "Tavish and Kormak 4: done"
	on offer
		conversation
			`The three of you ride the rail in silence as it slithers through Chiron's bright and beautiful landscape. Some minutes into the trip the obvious signs of the city give way to university campus scenery.`
			`	For an hour you pass one educational building after another, only broken up by the occasional lacrosse court and dormitory, until finally Tavish stands up.`
			`	"Next stop is us," he says.`
			choice
				`	"How big exactly is this campus?"`

> 			`	Tavish manages a chuckle, "You think this is big? You should see the Academy of Planetary Sciences campus on Glory."`

Author: I understand what you're saying, but the Academy of Planetary Sciences is described thusly:

> The Academy of Planetary Sciences is a campus the size of a small nation-state on Earth. Various artificially created and maintained microclimates abound as you travel between miniature mountains and micro-savannahs; between marshland and jungle, desert and forest. Impossible machines - some larger than skyscrapers - litter the twisted and ever-changing landscape, and terraforming satellites make meteor showers overhead in the dawn light. This is a place where anything is possible, and land bends eagerly to the whims of the terraforming wizards.`

> 			`	Tavish manages a chuckle, "You think this is big? You should see the Academy of Planetary Sciences campus on Glory."`

Reviewer 4: Huh, then go for it. Resolve this one as well

				goto gotime

			label crawl
			action
				clear "tavtalk"
				clear "payme"
				clear "tavgirl"
				clear "tavgirl"
			`	Tavish wilts when he hears your command, but he does not argue. Humbly he nods to you, "It's my problem to resolve."`
			`	Tavish's cheeks are wet as he drops to his knees. With painful awkward slowness he starts to bend over as to be on all fours.`
			`	Before he is on his hands and knees, he is stopped by Kormak who pulls his companion to his feet.`
			`	Kormak turns to you his face contorted in rage and says, "Bad friend."`
			`	Kormak then stalks toward Hullen-Freeze.`
			`	"Wa-wait! What is this? Stay back, savage! Starla! Starla, call secur..." Hullen-Freeze is abruptly cut off as a heavy fist slams into his face.`
			`	Kormak wastes no time, and holds nothing back. Hullen-Freeze is dead within a few seconds. Kormak searches through drawers and finds a key to Hullen-Freeze's equipment storage.`
			`	Kormak turns back to you a final time and says, "We find new Captain. Better Captain."`
			`	Dazed, Tavish holds onto Kormak's arm as he leads the two of them out of the office.`
			`	The last thing you see is Starla running after them.`
				decline

> 			label kortime

Author: I used your suggestion below. Is the better or do you think it needs more work?

> 			label kortime

Reviewer 4: I think probably fine then. It's about the same as some of the other longer chunks in here.

			`	Kormak grabs his jacket and finds Starla waiting for him just outside the office. She'd seen the whole thing.`
			`	"Wh-what's that smell?" she asks quietly.`
			`	Kormak looks back, sniffs, and then shrugs, "He wet himself."`
			`	Starla gasps, but it's unclear if she's horrified or impressed. She then presses something small in Kormak's hand.`
			`	"If you're ever on Chiron, give me a call." She then turns and hurries away.`
			`	Tavish, who has been silent up until now, suddenly bursts out in laughter. Long uproarious belly laughter.`
			`	The three of you leave together and Tavish laughs the entire way back to the spaceport. Once there you finally pick up the scientific research equipment for the New India expedition.`
				accept

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
	on complete
		conversation
			`You arrive in the dusty, rowdy frontier spaceport where it all began. Tavish lets you know that he and Kormak are going to pick up some supplies from a local outfitter and asks you to meet him in the spaceport later.`

> event "killhullen"

Author: I think it's currently used in the Weland PR.

			choice
				`	"The <ship>."`
			`	"Right, of course, a ship worthy of a captain of your caliber, no doubt, no doubt." Tavish tips his hat to you, and then he and Kormak make their way out of the cafe.`
				accept
			label forscience
			`	Tavish breaks into a wide grin and mock punches the air just above his head, "Huzzah, my good chap! Yes, that is the pioneering spirit! Just the news we needed to hear, for I must admit to you, that my esteemed colleague and I have run into to some trouble of the financial variety. We paid another very charming captain to pick up our equipment on Clark, and they never returned. We then paid another captain to take us there, but when we went to meet them at their ship we discovered the most unwelcome news that they in fact were not its owner."`
			`	"We are broke," Kormak helpfully adds.`
			`	"Ah yes," Tavish sighs. "Our most unfortunate circumstance, we are stranded here without money, and have yet to find anyone capable of setting aside their normal preoccupation with short-sighted material gains and instead recognize the more noble pursuit of pure scientific discovery!"`
			`	"We asked everyone." Kormak says.`

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`

Author: Hm, he isn't meaning it in a rude way. It's just an example of how Tavish likes to pepper his language with words that he thinks makes him sound smart.

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`

Reviewer 2: Combined with "venerable", I don't think it necessarily sounds nice; how about:
```suggestion
			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your sturdy dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`
```

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`

Author: Okay, but venerable also falls into his word use formula. Also venerable is _more_ positive than sturdy.

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`

Author: i could do venerable ark?
Although venerable dirigible has a real nice mouth feel

> 			`	"And yet we are not the least bit perturbed about our perilous predicament!" Tavish continues. "For such setbacks are to be expected in the pursuit of discovery. And now that we've found you, the famous captain, Captain... Mayflower? Our fortunes are already changing for the better! When you are ready escort us to your venerable dirigible, we shall then commence with our grand adventure to the hardscrabble world of Clark!"`

Reviewer 2: It works the way it is

			`	"We had a special mechanical ring made to surround our campsite in ignited heat and prevent the tree-analog's tendrils from entering or interfering with our work."`
			`	"Firewall," Kormak adds.`
			`	"There was also a drill and sifter specially designed and programmed by Dr. Gerald to help uncover the vessel without damaging it," Tavish continues, "and, lastly, an underground sonar headset that would allow us to see its buried structure and avoid injuring it during our excavation."`
			`	Tavish shrugs. "There were other things like tents, and picks, and shovels and what not. But all of that we can get on world. The ring, drill, and headset are all irreplaceable and absolutely necessary for this dig."`
			`	"Irreplaceable," Kormak confirms.`
				goto questions2

			label end
			action
				clear "who1"
				clear "india1"
				clear "equipment1"
			`	Tavish brightens as he says, "I knew an erudite fellow such as yourself would see the value of our work. And for now on we will check with you before we do anything too untoward, right, Kormak?"`
			`	The red-haired giant nods, "Ask first."`
			`	"Grand," Tavish claps. "Now, let's see what we can discover at <planet>-"`

> 			`"Hey! Stop right there!" a security guard shouts. You bolt onto the <ship> to launch before the authorities arrest all three of you.`

Author: What do you recommend? My thinking was that a minor altercation in a busy boom-town that didn't result in any serious damage wouldn't result in a thorough investigation. The authorities have too many other problems on their hands.

mission "Tavish and Kormak 3"
	name "Board the Broken Record"
	description "Find the <npc>, disable it, board it, and retrieve the stolen scientific equipment."
	source "Deadman's Cove"
	waypoint "Persian"
	destination "Hephaestus"
	passengers 2
	to offer
		has "Tavish and Kormak 2: done"
	on offer
		conversation
			`Hospitality is in short supply as most of the pirates and free traders you pass either ignore you entirely or give you menacing glares. The spaceport bar on the archipelago's main island functions as a sort of catchall for the many services you'd expect at a larger and better organized spaceport. Various fixers, vendors, mercenaries, and opportunists ply their trades in corners, booths, and tables scattered everywhere.`
			`	Tavish makes a beeline for one of the bartenders, the closest thing to any sort of identifiable administrator, and waves him down.`
			`	"What'll it be, your lordship?" the bartender asks mockingly after seeing Tavish's overly formal getup.`

> 			`	"I'll take a sidecar, my good man, and my large companion will have a Scottish Ale, assuming you have any," Tavish says brightly.`

Author: Added an option for the player to also get rum.

			`	"My good man, you seem like a discerning fellow. Always looking to make a deal with an eye out for profit, am I right?"`
			`	The bartender looks piqued and slowly nods, "Aye."`
			`	"You're probably wondering what a pompous overdressed man like me is doing in your quaint out-of-the-way boutique. Well, I was looking for just the sort of enterpising soul who knows a thing or two about making a shilling."`
			`	Tavish proceeds to describe a long and complex scam utilizing unwitting pirates, corrupt Syndicate trade officials, and fabulously wealthy marks from the Paradise Worlds. As he talks, the barkeep leans further and further in, his eyes widening beneath a growing devious grin."`
			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 30,000 credit buy in, plus the information on our first unwitting mark, and you're all set to be rich."`
			`	"I'm in!" The bartender can't move fast enough, and immediately begins transferring the funds. "Who are you looking for?"`
			choice
				`	"We're looking for Hans Fuller of the Broken Record."`
			action

> 				payment 30000

Author: They're broke. You're the money guy in this arrangement. Tavish is speech, Kormak is strength and you are bribe. It also is supposed to subtlety make you feel a little used and worried that Tavish might actually be conning you.

			`	"Sorry man, good luck with your science stuff or whatever. But that's all I know so I'm out of here."`
			`	Hans waits until you transfer the money over to him, and then quickly retreats deeper into his ship.`
			`	Tavish watches him go. "I dare say he didn't deserve additional recompense after what he did to us."`
			`	Kormak grunts in agreement.`
			launch

	on visit
		dialog `You haven't found the Broken Record yet.`
	on complete
		conversation
			`Your ship lands in the heart of the continent-consuming megatropolis Polyphron, the "only city" on Hephaestus.`
			`	Finding your way around here could well be impossible. Thankfully Tavish says he knows the area where recently impounded ships are housed.`
			`	He and Kormak head toward the spaceport.`

mission "Tavish and Kormak 4"
	name "A journey to Chiron"
	description "Transport Tavish and Kormak to <planet> to look for a man named Doctor-Professor Dylan Hullen-Freeze."
	source "Hephaestus"
	destination "Chiron"
	passengers 2
	to offer
		has "Tavish and Kormak 3: done"
	on offer
		conversation

> 			`You get the feeling the sun never sets on Hephaestus as you navigate through droves of itinerant workers all hurriedly moving from one shift change to next. The number of ships being worked on, landing, taking off, or just sitting is staggering. It would be easy to believe this was the beating heart of human expansion, rather than Earth.`

Author: I think that, given all the ships, many of them aren't being repaired? But upgraded, or tuned up, or just having diagnostics ran.

			choice
				`	"We're not here about Tavish's old position. We're looking for scientific equipment that was confiscated on Hephaestus."`
				`	"I don't care about you and whatever this thing you have going on with Tavish is. We're just here for the confiscated equipment from Hephaestus."`
			`	Hullen-Freeze narrows his eyes to peer at you as he continues to wipe his glasses, "What? Oh, that ecletic junk?"`
			`	He laughs, "Of course! Only Julian and his uncouth associates would have use for such nonsensical tools. A metal ring that blazes fire? Really? This is too good.`
			`	"You don't know who you're really working with, do you? I bet he told you he was some sort of hot-shot Chiron professor? He's not. He's street-garbage. A common thief and a conman. He thinks if he says a few big words and calls everyone 'chap,' they'll think he's important. But the only reason he isn't trash is because of me!"`
			choice
				`	"I don't care where Tavish came from, he's my friend. Now tell me where the equipment is!"`
				`	"Just shut up. No one cares what you think. Just tell me where the equipment is."`

> 				`	"Look. I am just here for the equipment. Tell me where it is and we'll go."`

Author: Done.

			`	"The Ui Skathi," Kormak says. "Granddaughters of Skathi."`
			`	"Was that what they called themselves?" Tavish shrugs. "Kormak's always a little sensitive about this. Anyways, they don't really make sense. How did they get to the Deep? Where did they come from?`
			choice
				`	(Say nothing.)`
				`	(Motion for Tavish to continue.)`
				`	"So where did they come from?"`
			`	There's two prevailing theories. First, that there was a lost generational fleet that landed in the Deep earlier than anyone realized, that something went wrong, and that they degenerated in primitives."`
			`	Kormak frowns, "Lies."`
			`	"Or, and this is more folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`
			`	"Not elves," Kormak interrupts with a hint of annoyance, "the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`
			choice
				`	"I think it may have been the alien race called the Pug who brought your descendants here."`
					goto thepug
					to display
						has "main plot completed"
				`	(Say nothing.)`
					to display

> 						has "main plot completed"

Author: Originally neither options were there, so I thought without the main mission completed it would flow as though there is no choice again?

> 			choice
> 				`	"I think it may have been the alien race called the Pug who brought your descendants here."`
> 					goto thepug
> 					to display
> 						has "main plot completed"
> 				`	(Say nothing.)`
> 					to display
> 						has "main plot completed"

Reviewer 2: I'm not sure - it's probably worth retesting. Here's the suggestion in text form:
```suggestion
			choice
				`	"I think it may have been the alien race called the Pug who brought your descendants here."`
					goto thepug
					to display
						has "main plot completed"
				`	"Do you know where these aliens might be?"`
					to display
						not "main plot completed"
				`	(Say nothing.)`
					goto saynothing
			`	"Nobody knows, although some would say the exact middle of eminent human space would suffice as a distinct hiding place," Tavish responds.`
			label saynothing
```
You could also implement this by hand, especially if you want Tavish to have a more Tavistic speech pattern.

			label thepug
			`	Tavish perks up, "Them again? How interesting. I wonder how long they have been meddling in the affairs of men."`
			`	"Maybe not lies." Kormak admits, and then looks down as if in deep thought.`
			`	"But maybe not whole story," He says finally.`
			`	The large man shakes his head and frowns. "Friend," He says the word slowly, carefully enunciating each syllable.`
			`	"If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf. Know that is my home."`
				goto pugout

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

event "Tavish and Kormak Expedition"
event "Tavish and Kormak Late"
event "Tavish and Kormak Abandoned"

mission "Tavish and Kormak 7"
	landing
	name "Alexandria awaits"
	description "Transport Tavish and Kormak and their cargo to <planet>."
	to offer
		has "event: Tavish and Kormak Expedition"
		not "event: Tavish and Kormak Abandoned"
		not "event: Tavish and Kormak Late"
	source "New India"
	destination "Alexandria"
	passengers 2

> 	cargo "Scientific Research Equipment" 90

Reviewer 2: ```suggestion
	cargo "Scientific Research Equipment" 90
	blocked "You do not have enough space to pick up Tavish and Kormak with their tons of cargo. Return here when you have 90 tons free."
```
Let's just hope `blocked` works on `landing` missions...

	on offer
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice

> 				`	(Land on Ashoka to pick up Travis and Kormak.)`

Reviewer 2: ```suggestion
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
```
~~I know, I dislike autocorrect too~~

			`	Tavish does not unload the large piece of alien equipment off your ship, and instead comes to speak with you.`
			`	"This is where we part ways, Captain <last>," Tavish says with a hint of sadness. "It was truly an adventure. Perhaps one day our paths will cross again."`
			`	Tavish sniffles for a moment. "Apologies, allergies. Anyways, old chap, I just wanted you to know that Kormak and I, we pay our debts. And though we have no money to offer you, we are instead bequeathing to you the large artifact we found. We scanned it pretty thoroughly and we don't believe it has additional scientific value. However, we think a collector would pay well for it. Quite well."`
			`	He leans a little closer to you, "The best place to look for that sort of commerce is Wayfarer."`
			choice
				`	"Thanks, Tavish. It's been grand."`
				`	"This better be worth it."`
			`	A tear edges its way down Tavish's nose. "Thanks again, Captain. And farewell."`
			`	Tavish turns and leaves you to rejoin Kormak and the excited hovering members of the Society of Gentlemanly Exemplary Exploration.`

> mission "Tavish and Kormak Late"

Reviewer 2: The other option would be to have branches based on whether you have the events; I'm not sure if you want to do that.

Author: I don't work with branches...

Reviewer 2: It works this way, but I'm also happy to send a PR to your branch turning it into a single branch mission.

Author: if no one minds the way it is, I see no reason to do extra work to make it a different way!

	landing
	name "Alexandria awaits"
	description "Transport Tavish and Kormak and their cargo to <planet>."
	to offer
		has "event: Tavish and Kormak Expedition"
		not "event: Tavish and Kormak Abandoned"
		has "event: Tavish and Kormak Late"
	source "New India"
	destination "Alexandria"
	passengers 2
	cargo "Scientific Research Equipment" 90
	on offer
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice

> 				`	(Land on Ashoka to pick up Travis and Kormak.)`

Reviewer 2: ```suggestion
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
```

					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka

> 			`	You land at the same stony outcrop as before, although now it's litered in various lean-tos and exotic animal offal.`

Reviewer 2: ```suggestion
			`	You land at the same stony outcrop as before, although now it's littered with various lean-tos and exotic animal offal.`
```

			`	A threadbare Tavish and Kormak approach you. They are skinnier than you recall, with sunken eyes.`
			`	"Were you apprehended by pirates, Captain <last>?" Tavish says, vitrol leaking into his voice.`
			`	"Or lost?" Kormak adds tersely.`
			choice
				`	"I was detained."`
					goto detained
				`	"I lost track of time."`
					goto forgot
				`	"I am here, aren't I? Let's go."`
			`	Tavish sighs loudly and then turns to oversee the loading up of the equipment.`
			`	Kormak gives you a hard look and then finally shrugs. "Glad to leave."`

> 			`	Tavish returns, "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 2: ```suggestion
			`	Tavish returns. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`
```

			`	"Hup! Hup! To Alexandria!" Tavish shouts with obvious excitement.`
			`	Kormak cracks his knuckles and grins, "Finally."`
				launch

			label detained
			`	Kormak's eyes narrow, "Monsters?"`
			`	Tavish shakes his head, "No, old chap, there's no way those wraiths could leave your world. Likely something much more mundane, isn't that right Captain <last>?"`
			choice
				`	"It was pirates."`
					goto pirates
				`	"I just forgot."`
					goto forgot
				`	"Got held up by bureaucracy."`
			`	Tavish nods slowly, while Kormak stares blankly at you.`
			`	"Indeed, mankind knows no greater enemy. But, we have survived, little worse for the wear, and our triumphant return is near! Now, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`
				goto owed

			label pirates
			`	Kormak grits his teeth, "Savages."`
			`	Tavish nods slowly, and says with great solemnity, "Then we shall not hold your tardiness against you. How cruel the slings of fate that our own myopic brethern retard our glorious progress."`

> 			`	"Cruel." Kormak agrees with a bloodthirsty grin. As though he's thinking of the cruel things he will be doing to pirates in the near future.`

Reviewer 2: ```suggestion
			`	"Cruel," Kormak agrees with a bloodthirsty grin, as though he's thinking of the cruel things he will be doing to pirates in the near future.`
```

			action
				outfit "Salvaged Artifact" 1
			`	Weland then launches into sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should you ever need a cord, and promptly leave.`
			`	Weland is a man of his word, and your Salvaged Artifact is soon returned to your ship.`
				decline
			label agree
			action
				event "welandwait2" 160 180
			`	Weland beams. "That's the spirit, now. Figurin' things out, well, that's what it means to be human, isn't it? Now, as I said, I reckon it'll take me near about 6 months to get this beauty figured out. I realize, being a fancy space hero an' all, you probably will need a little reminder, so when I am done I'll send you a message. As I promised, if I can't get it to work, you don't have to pay me a penny. But, well, wouldn't be betting on that happening; I haven't yet met a machine I couldn't crack."`
				decline

event "welandwait2"

> mission "Tavish and Kormak dead"

Reviewer 2: Maybe move this above the Weland missions?

Author: Done!

	landing
	to offer
		has "event: Tavish and Kormak Abandoned"
	source "New India"
	on offer
		log "People" "Tavish and Kormak" `Tavish and Kormak appear lost to the wilds of New India.`
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice

> 				`	(Land on Ashoka to pick up Travis and Kormak.)`

Reviewer 2: ```suggestion
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
```

					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka
			`	You land at the same stony outcrop as before. Beyond it you find scattered and abandoned equipment.`

> 			`	Nearby is the so-called 'ring of fire', blackened and corroded with the scraps of ruined tents at its center. There is sign no Tavish or Kormak have been here recently.`

Reviewer 2: ```suggestion
			`	Nearby is the so-called "ring of fire," blackened and corroded with the scraps of ruined tents at its center. There is no sign Tavish or Kormak have been here recently.`
```

			`	Tavish gives you a somber shrug. "Everything else is irrelevant details."`
				goto options
			
			label soggy
			action
				set "soggy"
			`	"It's a real organization, although it's been in decline for quite sometime. Back in its heyday it was one of the most prestigious clubs for scientists and explorers. But these days it's something of a joke in most circles." Tavish sighs. "I am just glad the esteemed Dr. Henry Forsyth isn't alive to see it in such a sad state."`
			choice
				`	"Dr. Henry Forsyth?"`
					goto henry
				`	"So, it's a joke?"`
			`	Tavish's face becomes very serious, "It's not a joke. Dr. Henry Forsyth saved millions, maybe billions, of lives by going alone to the unexplored reaches of a hostile world. They all thought he was crazy, that there could be nothing of value in that frozen desolate place. But then he discovered the Ice-Witches of Nifel, and beguiled them with his stories of civilization. Their leader, someone called the Ornery-dress, fell in love with him and unwittingly led him to the cure of the Rigellian Plague."`

> 			`	He seems to inflate as he speaks, "That's the legacy of SOGEE, of the Society of Gentlemanly Exemplary Explorers, and I may not ever live up to Dr. Forsyth, but I still believe in the spirit of his actions and am committed to following in his footsteps."`
> 			`	Tavish pauses. "Although, don't repeat that story to Kormak. It upsets him. I think the Ice-Witches of Nifel may be distant cousins of his."`
> 				goto options

Reviewer 3: We can simplify this as it is repeated in the next label
```suggestion
				goto henry2
```

			
			label henry

> 			`	"Dr. Henry Forsyth saved millions, maybe billions of lives, by ignoring the academic consesus of the day, and putting his life on the line by leading a daring expedition into the frozen wastes of a desolate world. There he discovered the barbaric Ice-Witches of Nifel, and seduced their wild and beautiful leader, the Ornery-dress. She would unwittingly lead him to the cure of the Rigellian Plague, and secure his position as a preeminent scholar, explorer, and hero of his time."`

Reviewer 3: ```suggestion
			`	"Dr. Henry Forsyth saved millions, maybe billions of lives, by ignoring the academic consensus of the day, and putting his life on the line by leading a daring expedition into the frozen wastes of a desolate world. There he discovered the barbaric Ice-Witches of Nifel, and seduced their wild and beautiful leader, the Ornery-dress. She would unwittingly lead him to the cure of the Rigellian Plague, and secure his position as a preeminent scholar, explorer, and hero of his time."`

			label henry2
```

			`	"Wh-what's that smell?" she asks quietly.`
			`	Kormak looks back, sniffs, and then shrugs, "He wet himself."`
			`	Starla gasps, but it's unclear if she's horrified or impressed. She then presses something small in Kormak's hand.`
			`	"If you're ever on Chiron, give me a call." She then turns and hurries away.`
			`	Tavish, who has been silent up until now, suddenly bursts out in laughter. Long uproarious belly laughter.`
			`	The three of you leave together and Tavish laughs the entire way back to the spaceport. Once there you finally pick up the scientific research equipment for the New India expedition.`
				accept

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
	on complete
		conversation
			`You arrive in the dusty, rowdy frontier spaceport where it all began. Tavish lets you know that he and Kormak are going to pick up some supplies from a local outfitter and asks you to meet him in the spaceport later.`

event "killhullen"

> mission "Tavish and Kormak 6"

Reviewer 3: Up to here for now

mission "Tavish and Kormak 7"
	landing
	name "Alexandria awaits"
	description "Transport Tavish and Kormak and their cargo to <planet>."
	to offer
		has "event: Tavish and Kormak Expedition"
		not "event: Tavish and Kormak Abandoned"
		not "event: Tavish and Kormak Late"
	source "New India"
	destination "Alexandria"
	passengers 2
	cargo "Scientific Research Equipment" 90
	blocked "You do not have enough space to pick up Tavish and Kormak with their tons of cargo. Return here when you have 90 tons free."
	on offer
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka
			`	Tavish and Kormak are waiting with all their equipment near the stony outcrop where you landed before. They seem more than a little relieved to see you.`
			`	"Knew we could count on you," Tavish happily says as he oversees the loading up of the equipment.`
			`	"Honorable," Kormak adds.`

> 			`	Tavish gives you a serious look. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 3: ```suggestion
			`	Tavish gives you a serious look. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```
The station is actually "Alexandria Station" iirc, it's just that "Alexandria Station" was too long to display on the map so MZ left it as "Alexandria". I think it's the same story with Mordente-Bridi, which is officially "Mordente-Bridi Station".

		has "event: Tavish and Kormak Late"
	source "New India"
	destination "Alexandria"
	passengers 2
	cargo "Scientific Research Equipment" 90
	on offer
		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka
			`	You land at the same stony outcrop as before, although now it's littered with various lean-tos and exotic animal offal.`
			`	A threadbare Tavish and Kormak approach you. They are skinnier than you recall, with sunken eyes.`
			`	"Were you apprehended by pirates, Captain <last>?" Tavish says, vitrol leaking into his voice.`
			`	"Or lost?" Kormak adds tersely.`
			choice
				`	"I was detained."`
					goto detained
				`	"I lost track of time."`
					goto forgot
				`	"I am here, aren't I? Let's go."`
			`	Tavish sighs loudly and then turns to oversee the loading up of the equipment.`
			`	Kormak gives you a hard look and then finally shrugs. "Glad to leave."`

> 			`	Tavish returns. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 3: ```suggestion
			`	Tavish returns. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```

			`	The final artifact is a huge lump of black and twisted alien wreckage that must weigh over 50 tons. It is being carried carefully by an army of strained robotic dollies that inch toward your vessel.`
			`	Tavish grins as he watches it get loaded. "The main ship itself was bricked. Old and mostly destroyed before it even hit the planet. But that piece broke off and was in slightly better shape. It's a little too big to show off to our fellows, but I do believe it will be worth a fair bit of credits to right patron. We can talk about that more in Alexandria."`
			`	With that everything and everyone is on board.`
			`	"Hup! Hup! To Alexandria!" Tavish shouts with obvious excitement.`
			`	Kormak cracks his knuckles and grins, "Finally."`
				launch

			label detained
			`	Kormak's eyes narrow, "Monsters?"`
			`	Tavish shakes his head, "No, old chap, there's no way those wraiths could leave your world. Likely something much more mundane, isn't that right Captain <last>?"`
			choice
				`	"It was pirates."`
					goto pirates
				`	"I just forgot."`
					goto forgot
				`	"Got held up by bureaucracy."`
			`	Tavish nods slowly, while Kormak stares blankly at you.`

> 			`	"Indeed, mankind knows no greater enemy. But, we have survived, little worse for the wear, and our triumphant return is near! Now, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 3: ```suggestion
			`	"Indeed, mankind knows no greater enemy. But, we have survived, little worse for the wear, and our triumphant return is near! Now, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```

				goto owed

			label pirates
			`	Kormak grits his teeth, "Savages."`
			`	Tavish nods slowly, and says with great solemnity, "Then we shall not hold your tardiness against you. How cruel the slings of fate that our own myopic brethern retard our glorious progress."`
			`	"Cruel," Kormak agrees with a bloodthirsty grin, as though he's thinking of the cruel things he will be doing to pirates in the near future.`

> 			`	"But enough about our ills, let us instead venture to focus upon our gains, Captain <last>," Tavish says brightly. "Old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 3: ```suggestion
			`	"But enough about our ills, let us instead venture to focus upon our gains, Captain <last>," Tavish says brightly. "Old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```

				goto owed

			label forgot
			`	Tavish's face droops into a profound frown. "I suppose we should take solace you remembered to return at all," he says unhappily.`
			`	Kormak glares, "At least honest."`
			`	Slowly Tavish nods, "I guess you're right about that, at least Captain <last> was honest with us, and it's not as though we've offered them any actual compensation. Perhaps the fault is ours."`
			`	"Friend, though," Kormak says.`
			`	"Indeed," Tavish answers, "We had hoped better of our friend."`
			choice
				`	"I am sorry."`
				`	"Whatever, let's get on with it."`
				`	(Say nothing.)`
			`	Tavish clears his throat, "There is no need for us to belabor the point. Captain <last> is here now, and we and our findings are intact. Let us move forward, to our impending triumph, and not backwards.`
			`	Kormak nods, "Forward."`

> 			`	Tavish gives you a wry smile. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`

Reviewer 3: ```suggestion
			`	Tavish gives you a wry smile. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```

		conversation
			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
			choice
				`	(Land on Ashoka to pick up Tavish and Kormak.)`
					goto Ashoka
				`	(Ignore the signal.)`
			`	You ignore the signal and land in the regular New India spaceport.`
				defer

			label Ashoka
			`	You land at the same stony outcrop as before. Beyond it you find scattered and abandoned equipment.`
			`	Nearby is the so-called "ring of fire," blackened and corroded with the scraps of ruined tents at its center. There is no sign Tavish or Kormak have been here recently.`
			`	As you work your way through the overgrown remains of their camp you discover a hastily written note.`
			choice
				`	(Read the note.)`
					goto read
				`	(Throw it away and leave.)`
			`	Whatever happened here, you decide it's better not to know.`
				launch

			label read
			``
			`	The trees. The trees. The trees. the trees.`
			`	thetreeswhisperthetreestrees.`
			`	treestreestreestobuildhousesofourflesh.`
			`	ttrestrestresreseresrssressssr`
			``
			`	There's no other clues to what happened to Tavish or Kormak.`
				flee

> mission "Weland"
> 	source "Wayfarer"
> 	on offer

Reviewer 3: This mission doesn't appear complete, we could add a couple things to it.
```suggestion
mission "Weland"
	name "Investigating the Salvaged Artifact"
	description "You dropped off the Salvaged Artifact Tavish and Kormak found on New India with Weland for him to investigate. Come back after a week or so to <planet> to see what insights he's found."
	source "Wayfarer"
	on offer
```

			choice
				`	(Tell him about the Salvaged Artifact.)`
					goto AA
				`	(Ask him about weapons.)`
			`	Weland's face turns to stone, and his eyes glare hard at you. "No weapons."`
			`	He limps a little closer to you. "Not anymore, at least. If you heard different, you heard wrong. Weland does not do weapons.`
			`	"Now," he says, leaning back, "There anything else I can help you with? Perhaps a cord? Everyone needs a cord."`
			choice
				`	(Tell him about the Salvaged Artifact.)`
			label AA
			`	Weland nods slowly as he listens to you recount Tavish and Kormak's expedition.`
			`	"Well, isn't that somethin'. Knew those boys would make it big one of these days. Sounds like they found dragon tech. That's a rare sight, alright."`
			`	Weland strokes his beard for a moment, and then says, "Listen here. Maybe I can help. I'll need to take a look at it. Stories are great and all, but I gotta put eyeball to metal to know if what you're talking about is wortha damn, and then even if it is valuable and if there is anything I can do about it."`
			choice
				`	(Agree to leave the artifact with Weland.)`
					goto agreetoweland
				`	(Refuse to leave the artifact with Weland.)`

> 			`	Weland holds up his hands. "Alright, now, just hold up, you really want to keep dragging around some useless cargo from port to port? I'm not going nowhere." Weland lifts up one of his mangled feet, there's some sort of chain directly embedded into his ruined achilles heel. "Not as if I could run, even if I wanted to."`

Reviewer 3: ```suggestion
			`	Weland holds up his hands. "Alright, now, just hold up, you really want to keep dragging around some useless cargo from port to port? I'm not going nowhere." Weland lifts up one of his mangled feet, where there's some sort of chain directly embedded into his ruined achilles heel. "Not as if I could run, even if I wanted to."`
```

			`	"So, you sure you want to pass up on this? That you can't give me a few days to look at your artifact?"`
			choice
				`	(Agree to leave the artifact with Weland.)`
					goto agreetoweland
				`	(Refuse to leave the artifact with Weland.)`
			`	Weland shrugs. "If that's how you feel. Well, before you go, any chance I could offer you some premium cords at crazy discount prices?"`
			`	Weland then launches into another sales pitch on the utility of many of his useless cords. You politely nod, and edge toward the door, before promising him you'll come to him first should you ever need a cord, and promptly leave.`
				decline
			label agreetoweland
			action
				event "welandwait1" 6
			`	The old man grins brightly. "Good. Good. I 'spect it'll take me about a week to get my bearings on this artifact of yours. I'll also be needing your ship's schematics, it'll help me figure out how it can be installed, assumin' you want that."`

> 			`	With the deal completed, you head back to the spaceport. You send the pertinent information about your ship to Weland, and have the outfit transferred to him as well. Hopefully, he and your artifact are still here when you return after a week.`

Reviewer 3: ```suggestion
			`	With the deal completed, you head back to the spaceport. You send the pertinent information about your ship to Weland, and have the outfit transferred to him as well. Hopefully, he and your artifact are still here when you return after a week.`
				accept
			to complete
				has "event: welandwait1"
				has "mission: Weland 2: active"
```

			`	The large man shakes his head sadly. "These chains aren't what keep me here. They have other ways to watch me, to stop me, to own me. These are just a symbol, a reminder, a cruel jest. I could cut them easy as cuttin' bread, but before I left the door they'd be all over me, and who knows what they'd do to me next."`
			`	He gives you a pained smile. "I know what you're thinking. Be a hero, help the old man, some such nonsense. This isn't that story, and you aren't that person. So, let it rest. These are my chains, and this is my problem, and one way, or another, I'll deal with it. So, is your curiosity sated? Can we return to the matter at hand?"`
				goto questions
			label dragon
			`	"Yep, Dragons. Well, not the kind from Earth stories, or, at least, Earth, probably. We find 'em in the Rim, the South, the Dirt Belt, bits and pieces of them, that is. Reckon they must have lived here more than a hundred thousand years ago. Most their tech is in ruins, broken, dilapidated, hard to piece much of it together. Suffice to say, they were big brutes, they had big space ships, and they've been dead for a helluva long time."`

> 			`	He looks over at your artifact, "And you, well, you found somethin' of theirs."`

Reviewer 3: ```suggestion
			`	He looks over at your artifact. "And you, well, you found somethin' of theirs."`
```

	on offer
		event "stone waiting most" 14
		fail

> mission "Tavish and Kormak 1"

Reviewer 2: `<destination>` will say `<planet>` in the `<system>` system. `<source>` is fine, but I think it'd be more natural for them to say `<planet>` instead of `<destination>`

> mission "Tavish and Kormak 1"

Author: Okay, I went back through it and changed it to planet.

			`	"Oh, how mortifing! I..." Tavish's voice drops off as he appears to be, finally, at a loss for words."`
			`	"Looked in wrong manifest." Kormak seems to be suffering none of Tavish's tongue-tied embarrassment.`
			`	"Er, yes, about that..." There's a pause before Tavish, his face red, finally says, "We may have been a tad bit desperate, and have been asking every captain who lands here, regardless of reputation."`
			`	"We asked everyone," Kormak repeats his comment from earlier.`
			`	"I must have gotten the flight manifest confused with yours and a different ship. I am truly apologetic for misleading you. To be frank, we have little idea who you are, Captain <last>."`
			`	"None," Kormak adds.`
			`	Tavish holds up a finger. "But, we can already tell you are a chap of quality, and the possesor of a keen curious mind. The sort of fellow we can put our absolute faith in. In fact, I see a lot of ourselves in you; truly we are kindred spirits."`
			`	"Like looking in mirror," Kormak says.`
			`	Tavish suddenly stands. "But we shall keep you no longer, and instead go gather our meager belongings and meet you at your ship, the, uh... what was it called again?"`
			choice

> 				`	"The <ship>."`

Author: Option added.

			`Clark's spaceport is overwhelmed with traffic, and you end up waiting an additional hour before they can finally find room for your ship on one of the outlying concrete pads.`
			`	Moments after touching down, Tavish and Kormak - or Kormak and Tavish - are making their way toward your ship's exit.`
			`	Tavish nods to you as they pass. "We're most apprehensive about the state of our equipment. We're headed to the SpaceFork storage facility in the spaceport. Meet us there."`
			`	"Apprehensive," Kormak echoes as he disappears out the hatch and onto Clark.`

mission "Tavish and Kormak 2"
	name "Tavish and Kormak"
	description "Transport Tavish and Kormak to <planet> to look for a man named Hans Fuller."
	source "Clark"
	destination "Deadman's Cove"
	clearance `Tavish convinces the spaceport to let you land "free of charge."`
	passengers 2
	to offer
		has "Tavish and Kormak 1: done"
	on offer
		conversation
			`The layout of the crowded spaceport is... slipshod, at best. It's robustly evident that its growing needs have consistently outpaced its capacity and materials. Dead ends, unfinished pathways, incorrectly labeled buildings and maps all conspire to slow your progress.`

> 			`	Despite the spaceport's best efforts, you successfully work your way to the SpaceFork storage facility, which has moved twice from earlier warehouses that each still bear its name, and is now housed in what is still known as the Pentacorps Mercantile Cooperative Structure.`

Reviewer 4: How about
```suggestion
			`	Despite the spaceport's best efforts, you successfully work your way to the SpaceFork storage facility, which has moved twice from earlier warehouses that each retain its name, and is now housed in what is still known as the Pentacorps Mercantile Cooperative Structure.`
```

		event "Tavish and Kormak Expedition" 60
		event "Tavish and Kormak Late" 91
		event "Tavish and Kormak Abandoned" 122
		conversation
			`The remote continent of Ashoka races up to meet you as your ship descends over the thick jungle canopy. The location for the coordinates Tavish gave you is densely forested, but you spot a relatively flat stony outcrop about a dozen miles from the site. You land your ship upon the stone.`
			`	Tavish and Kormak's research equipment was much more extensive than they had initially let on. There are nearly a dozen robotic dollies, a few mobile robotic drills and shovels, and a handful of computers, chemical sensors, and power generators.`
			`	They've also changed from their regular three piece suits into khaki pants and jackets with wide brimmed hats. Tavish wields a machete and Kormak has what looks like a heavily modified axe.`
			`	Kormak notices you looking and smiles. He presses a button on his axe's handle and the blade suddenly crackles with electricity.`
			`	"Equipment." He grins and walks to where the wall of trees begins.`
			`	Tavish turns to you, "Shall we go?"`
			choice
				`	(Go.)`
				`	"Let's go."`

> 			`	So long as you don't look too hard at the jungle it feels perfectly normal, like you're in a standard Earth biome. But the moment you concentrate on any one feature too long the alienness of the environment becomes unmistakable and unsettling. The trees here are nothing like the ones on the inhabited continent of New India. They are covered in silver scales and pockmarked with gray gasping orifices.`

Author: I was meaning gasping, like you're hearing them make weird breathy breathing noises. Which, when they start screaming, makes sense?

> 			`	So long as you don't look too hard at the jungle it feels perfectly normal, like you're in a standard Earth biome. But the moment you concentrate on any one feature too long the alienness of the environment becomes unmistakable and unsettling. The trees here are nothing like the ones on the inhabited continent of New India. They are covered in silver scales and pockmarked with gray gasping orifices.`

Reviewer 2: Hmm, in that case I think it might be better to switch the word order around, like so:
```suggestion
			`	So long as you don't look too hard at the jungle it feels perfectly normal, like you're in a standard Earth biome. But the moment you concentrate on any one feature too long the alienness of the environment becomes unmistakable and unsettling. The trees here are nothing like the ones on the inhabited continent of New India. They are covered in silver scales and pockmarked with gasping gray orifices.`
```
Your choice on this one, though.

			`	Kormak takes your hand in his. "May the stones guide you," he says, and then disappears back into the jungle.`
				decline

			label thepug
			`	Tavish perks up, "Them again? How interesting. I wonder how long they have been meddling in the affairs of men."`
			`	"Maybe not lies." Kormak admits, and then looks down as if in deep thought.`
			`	"But maybe not whole story," he says finally.`
			`	The large man shakes his head and frowns. "Friend." He says the word slowly, carefully enunciating each syllable.`
			`	"If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf, know that is my home."`
				goto pugout

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`

event "Tavish and Kormak Expedition"
event "Tavish and Kormak Late"
event "Tavish and Kormak Abandoned"

mission "Tavish and Kormak 7"
	landing
	name "Alexandria awaits"
	description "Transport Tavish and Kormak and their cargo to <planet>."
	to offer
		has "event: Tavish and Kormak Expedition"

> 		not "event: Tavish and Kormak Abandoned"

Author: Why?

> 		not "event: Tavish and Kormak Abandoned"

Reviewer 2: You already check for the absence of the "Tavish and Kormak Late" condition. "Abandoned" will only fire after "Late", so the "Abandoned" doesn't need to be checked.

			`	Kormak joins up with you and Tavish and the three of you head to your ship.`
				accept

			label rum
			`	He nimbly tosses another glass on the bar, pours caramel colored rum into it, and slides it across to you.`
			choice

> 				`	(Drink the Rum.)`

Reviewer 2: ```suggestion
				`	(Drink the rum.)`
```

					goto drink
				`	(Ignore the rum.)`
					goto shrug
			label drink

> 			`	You swallow the brown luke-warm brown liquid. It has a sugary taste and only lightly burns going down. Your belly is briefly warmed by it.`

Reviewer 2: ```suggestion
			`	You swallow the luke-warm brown liquid. It has a sugary taste and only lightly burns going down. Your belly is briefly warmed by it.`
```

				set "tavgirl"
			`	Tavish leans against her desk, "Starla, we're not pedestrian fans here to ogle your patron. No, we're members of the Society of Gentlemanly Explorers, and we've come at a time of the greatest dire need. I can already tell you're a keen observer of human behavior, and not some cog in the wheel. See us for what we are and make the liberating choice to let us through."`
			`	Tavish adds a few other flourishes about the great work they do here, and how her boss would be impressed with her ability to think for herself. Throughout all of this Starla's bored expression remained fixed.`
			`	Finally she sighs and holds up a hand and says, "Enough! You think you're slick? You think you're smart? Do you know how many people like you come through these doors every day? So certain whatever pretentious project they are working and a few flashy words are all it takes to get past me?" She scowls at him.`
			`	"You're lucky I am even willing to write your information down and pass it on. And I am starting to reconsider that offer." She crosses her arms.`
			`	Tavish holds up both hands in defeat and backs away.`
				goto talk

			label korgirl
			action
				clear "tavgirl"

> 			action
> 				clear "tavgirl"

Reviewer 2: (did I make two suggestions here?)
```suggestion
```

			`	The last thing you see is Starla running after them.`
				decline

			label streetgarbage
			action
				set "streetgarbage"

> 			`	"Work with him? Never." Hullen-Freeze sneers. "I used him. As is the wont of someone of my station."`

Reviewer 2: ```suggestion
			`	"Work with him? Never," Hullen-Freeze sneers. "I used him. As is the wont of someone of my station."`
```

					goto thepug
					to display
						has "main plot completed"
				`	"Do you know where these aliens might be?"`
					to display
						not "main plot completed"
				`	(Say nothing.)`
					goto saynothing

> 			`	"Extinct? Beyond the star lanes? Living amongst us? Nobody knows, although I recall a venerable member of SOGGE believing fervently that the exact center of eminent human space would be an ideal hiding space. Something about deep space gravitational currents, I believe," Tavish responds.`

Reviewer 2: ```suggestion
			`	"Extinct? Beyond the star lanes? Living amongst us? Nobody knows, although I recall a venerable member of SOGEE believing fervently that the exact center of eminent human space would be an ideal hiding space. Something about deep space gravitational currents, I believe," Tavish responds.`
```
Yep, this is much more in the style of Tavish then my suggestion. Good work

	destination "Alexandria"
	passengers 2
	cargo "Scientific Research Equipment" 90
	blocked "You do not have enough space to pick up Tavish and Kormak with their tons of cargo. Return here when you have 90 tons free."

> 	blocked "You do not have enough space to pick up Tavish and Kormak with their tons of cargo. Return here when you have 90 tons free."

Reviewer 2: ```suggestion
```

			`	A tear edges its way down Tavish's nose. "Thanks again, Captain. And farewell."`
			`	Tavish turns and leaves you to rejoin Kormak and the excited hovering members of the Society of Gentlemanly Exemplary Exploration.`

> mission "Tavish and Kormak Reminder"
> 	landing
> 	on enter
> 	on enter
> 	to offer
> 		has "event: Tavish and Kormak Late"
> 	on offer
> 		conversation
> 			`A forgotten bowler hat tumbles out of a corner after your last jump. You recognize it has one of the hats Tavish used to wear. Odd that you haven't seen it until now. Perhaps it's a sign you should check on him at New India. Or perhaps it's nothing at all.`
> 			decline
> 

Reviewer 2: ```suggestion
mission "Tavish and Kormak Reminder"
	landing
	invisible
	to offer
		has "event: Tavish and Kormak Late"
	on enter
	on enter
	on enter
		conversation
			`A forgotten bowler hat tumbles out of a corner after your last jump. You recognize it has one of the hats Tavish used to wear. Odd that you haven't seen it until now. Perhaps it's a sign you should check on him at New India. Or perhaps it's nothing at all.`
				fail

```

	on visit
		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
	on complete

> 	log "A collector on Wayfarer may be interested in buying the Salvaged Artifact."

Reviewer 2: ```suggestion
		log "A collector on Wayfarer may be interested in buying the Salvaged Artifact."
```

		has "event: Tavish and Kormak Abandoned"
	source "New India"
	on offer

> 		log "people" "Tavish and Kormak" `Tavish and Kormak appear lost to the wilds of New India.`

Reviewer 2: ```suggestion
		log "People" "Tavish and Kormak" `Tavish and Kormak appear lost to the wilds of New India.`
```
